From f85509e8a90f4e7970524728c2fc99ebda7f11ed Mon Sep 17 00:00:00 2001
From: John Howard <howardjohn@google.com>
Date: Thu, 3 Mar 2022 15:26:39 -0800
Subject: Fix release builder fetch (#37696)

* Fix release builder fetch

* Fix license mirror

* Fix security test

* Regenerate certs
---
 .../{qtls-go1-17 => qtls-go1-18}/LICENSE      |  0
 pilot/pkg/xds/debuggen.go                     |  4 ++
 .../framework/tools/featuresgen/cmd/root.go   |  2 +
 prow/release-commit.sh                        |  3 +-
 .../security/file_mounted_certs/main_test.go  | 17 +++++--
 .../file_mounted_certs/p2p_mtls_test.go       |  2 +-
 tests/testdata/certs/default/cert-chain.pem   | 32 ++++++------
 tests/testdata/certs/default/key.pem          | 50 +++++++++----------
 tests/testdata/certs/default/root-cert.pem    | 34 ++++++-------
 tests/testdata/certs/dns/cert-chain.pem       | 30 +++++------
 tests/testdata/certs/dns/key.pem              | 50 +++++++++----------
 tests/testdata/certs/dns/root-cert.pem        | 34 ++++++-------
 tests/testdata/certs/generate.sh              | 10 ++--
 .../certs/mountedcerts-client/cert-chain.pem  | 36 ++++++-------
 .../certs/mountedcerts-client/key.pem         | 50 +++++++++----------
 .../certs/mountedcerts-client/root-cert.pem   | 34 ++++++-------
 .../certs/mountedcerts-server/cert-chain.pem  | 36 ++++++-------
 .../certs/mountedcerts-server/key.pem         | 50 +++++++++----------
 .../certs/mountedcerts-server/root-cert.pem   | 34 ++++++-------
 tests/testdata/certs/pilot/ca-cert.pem        | 34 ++++++-------
 tests/testdata/certs/pilot/ca-key.pem         | 50 +++++++++----------
 tests/testdata/certs/pilot/cert-chain.pem     | 30 +++++------
 tests/testdata/certs/pilot/key.pem            | 50 +++++++++----------
 tests/testdata/certs/pilot/root-cert.pem      | 34 ++++++-------
 24 files changed, 359 insertions(+), 347 deletions(-)
 rename licenses/github.com/marten-seemann/{qtls-go1-17 => qtls-go1-18}/LICENSE (100%)

diff --git a/licenses/github.com/marten-seemann/qtls-go1-17/LICENSE b/licenses/github.com/marten-seemann/qtls-go1-18/LICENSE
similarity index 100%
rename from licenses/github.com/marten-seemann/qtls-go1-17/LICENSE
rename to licenses/github.com/marten-seemann/qtls-go1-18/LICENSE
diff --git a/pilot/pkg/xds/debuggen.go b/pilot/pkg/xds/debuggen.go
index ce0d39dcc2..6c9b17e804 100644
--- a/pilot/pkg/xds/debuggen.go
+++ b/pilot/pkg/xds/debuggen.go
@@ -88,6 +88,10 @@ func (dg *DebugGen) Generate(proxy *model.Proxy, push *model.PushContext, w *mod
 	updates *model.PushRequest) (model.Resources, model.XdsLogDetails, error) {
 	res := model.Resources{}
 	var buffer bytes.Buffer
+	if proxy.VerifiedIdentity == nil {
+		log.Warnf("proxy %s is not authorized to receive debug. Ensure you are connecting over TLS port and are authenticated.", proxy.ID)
+		return nil, model.DefaultXdsLogDetails, fmt.Errorf("authentication required")
+	}
 	if w.ResourceNames == nil {
 		return res, model.DefaultXdsLogDetails, fmt.Errorf("debug type is required")
 	}
diff --git a/pkg/test/framework/tools/featuresgen/cmd/root.go b/pkg/test/framework/tools/featuresgen/cmd/root.go
index 657edd98a3..75630c1ac3 100644
--- a/pkg/test/framework/tools/featuresgen/cmd/root.go
+++ b/pkg/test/framework/tools/featuresgen/cmd/root.go
@@ -137,6 +137,7 @@ func readVal(v interface{}, path []string) []string {
 }
 
 func removeDashAndTitle(s string) string {
+	// nolint: staticcheck
 	return strings.Title(s[1:])
 }
 
@@ -147,6 +148,7 @@ func createConstantString(path []string) string {
 	for i := 0; i < len(path); i++ {
 		namePart := alphanumericRegex.ReplaceAllString(path[i], "")
 		namePart = replaceDashRegex.ReplaceAllStringFunc(namePart, removeDashAndTitle)
+		// nolint: staticcheck
 		namePart = strings.Title(namePart)
 		name += namePart
 		name += "_"
diff --git a/prow/release-commit.sh b/prow/release-commit.sh
index c8da4d92d3..49cbdb28ab 100755
--- a/prow/release-commit.sh
+++ b/prow/release-commit.sh
@@ -91,8 +91,7 @@ EOF
 # "Temporary" hacks
 export PATH=${GOPATH}/bin:${PATH}
 
-# cd to not impact go.mod
-(cd /tmp; go get "istio.io/release-builder@${BUILDER_SHA}")
+go install "istio.io/release-builder@${BUILDER_SHA}"
 
 release-builder build --manifest <(echo "${MANIFEST}")
 
diff --git a/tests/integration/security/file_mounted_certs/main_test.go b/tests/integration/security/file_mounted_certs/main_test.go
index 9cc0f9e5eb..d44951a620 100644
--- a/tests/integration/security/file_mounted_certs/main_test.go
+++ b/tests/integration/security/file_mounted_certs/main_test.go
@@ -19,12 +19,14 @@
 
 import (
 	"context"
+	"fmt"
 	"os"
 	"path"
 	"strings"
 	"testing"
 
 	v1 "k8s.io/api/core/v1"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"istio.io/istio/pkg/test/env"
@@ -165,10 +167,10 @@ func setupConfig(_ resource.Context, cfg *istio.Config) {
     pilotCertProvider: "mycopki"
   pilot:
     env:
-      # We need to turn off the XDS Auth because test certificates only have a fixed/hardcoded identity, but the identity of the actual 
-      # deployed test services changes on each run due to a randomly generated namespace suffixes. 
+      # We need to turn off the XDS Auth because test certificates only have a fixed/hardcoded identity, but the identity of the actual
+      # deployed test services changes on each run due to a randomly generated namespace suffixes.
       # Turning the XDS-Auth ON will result in the error messages like:
-      # Unauthorized XDS: 10.1.0.159:41960 with identity [spiffe://cluster.local/ns/mounted-certs/sa/client client.mounted-certs.svc]: 
+      # Unauthorized XDS: 10.1.0.159:41960 with identity [spiffe://cluster.local/ns/mounted-certs/sa/client client.mounted-certs.svc]:
       #    no identities ([spiffe://cluster.local/ns/mounted-certs/sa/client client.mounted-certs.svc]) matched istio-fd-sds-1-4523/default
       XDS_AUTH: "false"
 `
@@ -217,9 +219,14 @@ func CreateCustomSecret(ctx resource.Context, name string, namespace namespace.I
 
 	_, err = kubeAccessor.CoreV1().Secrets(namespace.Name()).Create(context.TODO(), secret, metav1.CreateOptions{})
 	if err != nil {
-		return err
+		if kerrors.IsAlreadyExists(err) {
+			if _, err := kubeAccessor.CoreV1().Secrets(namespace.Name()).Update(context.TODO(), secret, metav1.UpdateOptions{}); err != nil {
+				return fmt.Errorf("failed updating secret %s: %v", secret.Name, err)
+			}
+		} else {
+			return fmt.Errorf("failed creating secret %s: %v", secret.Name, err)
+		}
 	}
-
 	return nil
 }
 
diff --git a/tests/integration/security/file_mounted_certs/p2p_mtls_test.go b/tests/integration/security/file_mounted_certs/p2p_mtls_test.go
index 1df4d50fef..8d6a939b8d 100644
--- a/tests/integration/security/file_mounted_certs/p2p_mtls_test.go
+++ b/tests/integration/security/file_mounted_certs/p2p_mtls_test.go
@@ -41,7 +41,7 @@
 	ClientCertsPath  = "tests/testdata/certs/mountedcerts-client"
 
 	// nolint: lll
-	ExpectedXfccHeader = "By=spiffe://cluster.local/ns/mounted-certs/sa/server;Hash=865a56be3583d64bb9dc447da34e39e45d9314313310c879a35f7be6e391ac3e;Subject=\"CN=cluster.local\";URI=spiffe://cluster.local/ns/mounted-certs/sa/client;DNS=client.mounted-certs.svc"
+	ExpectedXfccHeader = `By=spiffe://cluster.local/ns/mounted-certs/sa/server;Hash=8ab5e491f91ab6970049bb1f032d53f4594279d38f381b1416ae10816f900c15;Subject="CN=cluster.local";URI=spiffe://cluster.local/ns/mounted-certs/sa/client;DNS=client.mounted-certs.svc`
 )
 
 func TestClientToServiceTls(t *testing.T) {
diff --git a/tests/testdata/certs/default/cert-chain.pem b/tests/testdata/certs/default/cert-chain.pem
index 2351878318..818f37ed66 100644
--- a/tests/testdata/certs/default/cert-chain.pem
+++ b/tests/testdata/certs/default/cert-chain.pem
@@ -1,20 +1,20 @@
 -----BEGIN CERTIFICATE-----
-MIIDMjCCAhqgAwIBAgIUaAVaH1ssljSp51Hm0bAEJUVf4dMwDQYJKoZIhvcNAQEL
-BQAwGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDAgFw0yMDAzMTMwMDUwNTBaGA8y
-MjkzMTIyNzAwNTA1MFowGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDCCASIwDQYJ
-KoZIhvcNAQEBBQADggEPADCCAQoCggEBAMrCr7ipMuR/Wm9tk3LXCHEHVci6PE9H
-gqQzSqsi/yb5EZPlhChDKcBZMi6XqcR06nP2L2E8LbgeYrEVb1K5cMStN5eIfT6w
-ZGRa4prDUx0DVyn0OEj+QDQsDC1t0XBh+HIXhg2cBdkUJhoHAyya+S80FoSfhULz
-UYNSk2v2Sg2ZmKZuF1Kzmh2K5RP2iWfSTyrxbY2hSw/Nlt4Ww/+1sUbyE774J374
-NjwhhyZJYwtxu96ZeYHIyB4QuP0tKBFH28F3clzlo8oLDbbT2pTL1vts9IGSboEN
-rm7gWTpCFsFpG0YB6FX9JvfoZ7gXB4fMSrgLnGnFD0229IbM1/PJqO8CAwEAAaNy
+MIIDMjCCAhqgAwIBAgIUFad49XzcT8c0W+ltJRmkcmncz5YwDQYJKoZIhvcNAQEL
+BQAwGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDAgFw0yMjAzMDMyMjM2NTJaGA8y
+Mjk1MTIxNzIyMzY1MlowGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDCCASIwDQYJ
+KoZIhvcNAQEBBQADggEPADCCAQoCggEBAKqM44S7MWaIPSZhOuRZvbjmeti6+L4n
+bGTncXs0suxbAJte+imklBgQY6ZDDnWwoylI2c0pzZ49bdDX1PHoLI9OJ8fIhqnn
+uLTosE137eZH33goeK8WAdF9rCachfUrVAzzDhYe3urVLlew4dvh+qXqG4muJm/I
+RU1IoQOS0Xy+nMEtit5YuMdDa5bWXvV0Qp6MdTxw9S2B7xeayQi6xzmTdB9UtpSd
+C/bdlEbtItZiRrTyzNLPAsXnUukf0l3loKuJiI6ToHe580AaXRp95g4PzZc2a/k8
+1g6NtosaPsbPkASnnlYh1ZfjM9Cusl2zl6hrkMoCNi5JUyXPOSxMMdkCAwEAAaNy
 MHAwCQYDVR0TBAIwADALBgNVHQ8EBAMCBeAwHQYDVR0lBBYwFAYIKwYBBQUHAwIG
 CCsGAQUFBwMBMDcGA1UdEQQwMC6GLHNwaWZmZTovL2NsdXN0ZXIubG9jYWwvbnMv
-ZGVmYXVsdC9zYS9kZWZhdWx0MA0GCSqGSIb3DQEBCwUAA4IBAQBGw4MudbJmx5Bb
-5pYm60R5keZB4PfGqNq2NxFNsj0YM9wrtH1nndRkLoS5nSYSlFSIiHmpbj/YhuWa
-g+WqUI+h8bml9kzUEBAFLPD2/3zsZtO6yGdUcNyVP7FZlCGU3ylCCHrw/xn6cMdf
-+SzCAk6NrY/z93aDcFqH+GVymrWdEeEfm3WKJISMtBIcxoKamrWI0TPziX7darPV
-gAuXCFMGogCroMnB4mL9y6UBU9jw1fuC63NHv5uNEdP6FEisV240thQ9RBmI8HSI
-MEhMqHAGwqTSFzSeMuHI1fhlEJ/TPhACjhYm8ag87zMK7dOKJgyNPmjJLpwbo3y8
-sH2Vo/2b
+ZGVmYXVsdC9zYS9kZWZhdWx0MA0GCSqGSIb3DQEBCwUAA4IBAQA79dKVl0EwiYhz
+Yuc/S058xsxOSpBsM3Mfca1li78/iSiZT4A0bjLxEE5Srss+GHM+OLFXoy3TjEix
+8ZV4LsiiYkIYVhiZCqWQEigz8fTnIiBLpljOIv0aPzCkeKSMcPEnKC/3UwZWToKF
+fhTl8dHsGpVsc/fLIE0qsGkUYsDKlgUTfcj6rUL3LHttLZJulCQmFZnrhNpejEjw
+oJ117b8RODMDv8NoJo9aDIxCA935e3JzNjBE37QSd932zaurEME+gG/121AHt/SQ
+3h/lxhIcSkEeJTTOjlroeFRm3hA1dJD0LwfDaykX8fTD1vp+krFj5IAZixrJ1HmP
+MpW4whMD
 -----END CERTIFICATE-----
diff --git a/tests/testdata/certs/default/key.pem b/tests/testdata/certs/default/key.pem
index 39ddb1ee14..f5f9cc45b9 100644
--- a/tests/testdata/certs/default/key.pem
+++ b/tests/testdata/certs/default/key.pem
@@ -1,27 +1,27 @@
 -----BEGIN RSA PRIVATE KEY-----
-MIIEpQIBAAKCAQEAysKvuKky5H9ab22TctcIcQdVyLo8T0eCpDNKqyL/JvkRk+WE
-KEMpwFkyLpepxHTqc/YvYTwtuB5isRVvUrlwxK03l4h9PrBkZFrimsNTHQNXKfQ4
-SP5ANCwMLW3RcGH4cheGDZwF2RQmGgcDLJr5LzQWhJ+FQvNRg1KTa/ZKDZmYpm4X
-UrOaHYrlE/aJZ9JPKvFtjaFLD82W3hbD/7WxRvITvvgnfvg2PCGHJkljC3G73pl5
-gcjIHhC4/S0oEUfbwXdyXOWjygsNttPalMvW+2z0gZJugQ2ubuBZOkIWwWkbRgHo
-Vf0m9+hnuBcHh8xKuAucacUPTbb0hszX88mo7wIDAQABAoIBAQDG0zxO/xPZvwxm
-YfQJ09UCSSgUqpMDRuNTsrxQrwO3KtvSxGjgtmOVq6Gq0Kv36pIRXmK9na5TcvBi
-J9u/Kfrr0dPGs1/lfkEjq3+0NT8mHyV1DXe4FCHPRzY1wg9zfY6MCoc4hzuwz7JB
-KYkyT1F0uUYvjOG3cjQHa40GfD/HroxuA0hq0IL+yak302chcMsj2Ms1KDI+I6Zs
-1/wOWYcG3cTlFj45EQDvAxouyAbakQOMDa/Pn4IC1d7y4PHtthXeqzP1gMAc21pC
-ix5rhsxA/Z+N2KLCZxvkC0V8u1y491oqdltUTk+WedEi+jrlhr8H1d8STjxRYnN/
-asBmutLBAoGBAOl2br3TGWcek6dASBx7+euazdJFvE1Mq8/gGL7LSd0Z9j9+WMTL
-cDAt/2vAkoLR0DnKKlISruQqC0n18F6jpUVk1cIurhjk2Wkx//Kom6HJqDlqxoLn
-Z+XW/uZ1Tr+Td5FK9mv3yauMpX00yIvcGOlYLw28J2HPsB1kP5ldpaWPAoGBAN5V
-guMh1bq+ozSYXEknbEIhg27+DwDynJ1LelBncIs1UWllPQoebIBpmjNzP925oB0K
-urzUhRmZA7e5aYCgiEUehT5UQbGuJeIEQjOPEKW4qI+v15PfDHstXnj2s36+Cd+q
-pc29ii77/Q8SA6MzM3nKgR7M/By4pBTufonA2dahAoGBAMUsZz8e6Nd0tZON8NlN
-w7e+mJI8n1pRpNH+HCgXnRyFbroVRXd/shKe5pfknliqR0hne/KxINBGyh3+/rGP
-WGgRT4+eyR7xOMVlmxsgV8D+hXtouJuaQN+XAtU0wft+2n71ZsWtEsA2GxZx2+Zz
-Tv7UoXjiLgin1cAdCLHIq7v1AoGBAKDaEMWuluhyP7UvfaTXRrVZdbN5TvWKkNAj
-6Sv+iifMvUeZHN4TQMBupkcO3GfXggVA2/T9dAYe89eZ5sBOZAhNmmyLnLOloHay
-SgrvOEYNILZDq52pw2fiXPT61sWdS5tr5GTC+hQf7udWSxdpVZaJcX0NJGDLHLqg
-CSA0S+XBAoGALkbcFKoX95eM6xQz3UTByIz+iJ1S4goZKy3TERJTJLQBqnxajGl0
-zVquR7iVCmig+gLO8dtDRCSTYBIVJ7c8ZuZ949RA8Pf8rvr//M1OkTYsmBML5yfy
-0C8ikYEkHcpWWzvpUa9nt7bIJXy9f13RQLCzDhX8z8tdLedg6VnZSlc=
+MIIEpgIBAAKCAQEAqozjhLsxZog9JmE65Fm9uOZ62Lr4vidsZOdxezSy7FsAm176
+KaSUGBBjpkMOdbCjKUjZzSnNnj1t0NfU8egsj04nx8iGqee4tOiwTXft5kffeCh4
+rxYB0X2sJpyF9StUDPMOFh7e6tUuV7Dh2+H6peobia4mb8hFTUihA5LRfL6cwS2K
+3li4x0NrltZe9XRCnox1PHD1LYHvF5rJCLrHOZN0H1S2lJ0L9t2URu0i1mJGtPLM
+0s8CxedS6R/SXeWgq4mIjpOgd7nzQBpdGn3mDg/NlzZr+TzWDo22ixo+xs+QBKee
+ViHVl+Mz0K6yXbOXqGuQygI2LklTJc85LEwx2QIDAQABAoIBAQClHdvUEEtGghX5
+x15137A/h9xdYb8akh5uHmiOqLmk/IG86pn1oSHDnnJrcDZamNWOTbqX16X7Umu0
+KpE+VtJ1N2xqDmjZq772XuT5S2fmb6UZts0IbZz7DdmSVC2Y7AbmtpXnsqGM5zGD
+snsoH0NESD+67wDFbjYyGWPRit4IDwntj31j2cI1ZpOpHma5OaSPpv384Mf+yU33
+HJHLZ/r1iEEKR9EzU4fTaL9JxYtMt1HAxVgRZAsfHiMBmArC6X+1dzriOtoKEQct
+sktVU4otEbWXQUw7qTEDN42O5ZMRgV46IVA9UwdzqeY6+qRzZZ5Crp4FbM+nFmAu
+0yx35gwBAoGBANgysX2qvozfoBBGg8buzdNZSNHbvykb9zMs9Kwa8npQcNgpg9/C
+OzTV3KITScJ0DXxt6oebqe2vrED+V0VgyTdL+8D2wEvq37Bf8JLdult4N9evaaj7
+dRy3C9eI4MdtfUg03fNGjys7Gw5coAK/STrwsF90J/gAo8V3R16ZPviBAoGBAMny
+18ohekO6tEEBJQvdYaGTEFV81nYYFL5DXM1eMvxHR+dthNfLKT2OE5JYrHhzRDvV
+s0BFF61J3suJSKb140Pf4w5zAnV9eM4ivAdlvbqzfcYCXnlm5TxBWayIRibMkmfZ
+eRW/sS5cLeHy24gpt/B/LUOUjIiWWybdH1QoKU1ZAoGBAK0IRY3ybfeJsy3Tgfxh
+fFI/D00GtwRL4ZkPzfNAzfifEPGwqF8ZsREkaSU+krYS9Ag2YJKQP3yAMTkDVw2r
+3N9o7OEPm/Am10juQDkLRO+YU/lH5yTbvGtzBQNIBb8XHkIALfNa8ZMEBy+Dan1I
+Y2/M2xxKdwLCYFDWmkRYGlIBAoGBALvPmboE+mUEvOB7hQ2MvGB8mcM3T2gSqy/r
+C+J+iaIvVIK5SsM50MfLWw5OY0+9uyZ+bq8F0qg5O/dEXInRwaL9E3f7o7zKjNAA
+Y6fpidKZXQ4KY/w5olUwXYpkTgGFXV8xVDfSx5/a4Fzy9BGk4sjKLblc7HmWg30Y
+RiuJEig5AoGBALquYHqGp/F4/cT/2JgimeKE/p5OSnVE53RM0qmpS+AoZQ0kfRYE
+VIj66AW4F/nIGfn4NB8WL2rk7Ebq7vzPYeEk06uN7kesz+J8IUfXVIr2zDDDq9NL
+/FEHkVYdg3SnOhw13zmycK8e5QLx/J6h+LTNGEKg0Bkvk9ZrhrEPrZMV
 -----END RSA PRIVATE KEY-----
diff --git a/tests/testdata/certs/default/root-cert.pem b/tests/testdata/certs/default/root-cert.pem
index 71062b7f8c..6f9c872974 100644
--- a/tests/testdata/certs/default/root-cert.pem
+++ b/tests/testdata/certs/default/root-cert.pem
@@ -1,19 +1,19 @@
 -----BEGIN CERTIFICATE-----
-MIIDEzCCAfugAwIBAgIUHu+RNpObSStNAQtgGqFPV70CNVEwDQYJKoZIhvcNAQEL
-BQAwGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDAgFw0yMDAzMTMwMDUwNTBaGA8y
-MjkzMTIyNzAwNTA1MFowGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDCCASIwDQYJ
-KoZIhvcNAQEBBQADggEPADCCAQoCggEBAN77xnr0H71SuBEGBI0amTmlvjwcgqZF
-mAw5c2pOCs3eUiJSz3tQYTRfVzaWBOrwEmn7RrQH8LJ5EVQm7Prg+aCT7koeBQyD
-G/Oqk2hpX1ArCLnjwEildUOlDSg+KBNJiImRFhMb2vnlFPfzkO5TsUk01OOnQTYm
-5XMzhQ0CuqX3stcAACsuKGxk3BX3r5zHmcFH4nBpbh9pb0DY2QZ+N+B6Qxe6o89q
-bT8gtWYLN1b3yGhn2WGeFkQtaud36Wj2jRSxWo2PBe8xc8qDBFJOt0zHg+u0Pt7t
-3mCrBIZ45FWEqX06FOupSe7HF2d7U/9pJGSZLe9iAJ/59WROlniJ7PUCAwEAAaNT
-MFEwHQYDVR0OBBYEFC1YD3F9RMLaW1xMuuQN27SmXzsMMB8GA1UdIwQYMBaAFC1Y
-D3F9RMLaW1xMuuQN27SmXzsMMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQEL
-BQADggEBAGqzLzTnusoOPF+tQ4jbhcXDCMvpDCCiAQVkkaorNGMWg2ny39vm3ymc
-HBhqWNNbeyZTj8hl7ffCoeUChZnIFkLO1ev3DVbLz179IP6dhqfDkuLhc/0g5Lfs
-FcNedAJh5TConI+x7dH1W2opPX4hXiiJBzcpPmfttD1EYES2VH85cPp95goUJwRp
-kMPAS5WiBLPv0c+0Msppe4iqxwuZZtJKRoHRDbvrHonMT1FmdOoEJ8xjeTxB8pjc
-tp5dA30dGNAxB7tOrzqmIidjXjYaxZGXKoC6BViw9ISfX9WNqVdvZVMJ7XwL6gu5
-Dk0yYXcMuGwMkMz+vJpypIBZAaCFES8=
+MIIDEzCCAfugAwIBAgIUf7oS8NPqTONgWOz1s8w4I6QWrv8wDQYJKoZIhvcNAQEL
+BQAwGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDAgFw0yMjAzMDMyMjM2NTFaGA8y
+Mjk1MTIxNzIyMzY1MVowGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDCCASIwDQYJ
+KoZIhvcNAQEBBQADggEPADCCAQoCggEBALD7cm5AqNsXTiAem/Wx2H26dYreh5OV
+g5DtTkAbN7lfoT98t9MFP5cgmzHcAKfaQ0RST+7tkp8j0IHgvZtvJYHAYCNjqXbo
+uKqgH2kdPDAjryWtUIk95md9ev91W50LKEEbMzxmBTTdsNoZ7HDuCYVSfenogxzV
+Uji4LheGcSvnDZ5QNvoog79Smn7m0qgEWXZmeD4pbQ5phRTE2Jz3VzTszr5Mzi7m
+9zPEvR8MLQcggVnWmHdCawVPGZrBbWfAtJXAFPNwNT9I9OXD2QubmEjv4IgTnxvw
+7fo+13CXKD2p8FYYpyZRaBwT/w9EcyEq5TA+HslWm5uwqi6Ezn9kgGcCAwEAAaNT
+MFEwHQYDVR0OBBYEFNPm0Yesg8sMVK3OhDMCIGW5FumoMB8GA1UdIwQYMBaAFNPm
+0Yesg8sMVK3OhDMCIGW5FumoMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQEL
+BQADggEBAFXlSpQZKB3lrcE+hbBfP4ToamIMv6Jn41tlPm0saV1Wqr+p3pAidRlu
+tEZEVC20V5IUVT/k3BGZLtPmrQQHfHHVNeQNwfPQpa/DeVWdSoXSfPxFiv4nIDEy
+orYGse/TDWGloID5NkDIz/WKSBIF0lurKnScC0sBJxyyW3UvTf4Fkf+fIS3ovqFN
+2k0zs9DRPu1NReuRc+BxHqCNq1z+mjmK7Z2aSDl7rSox1tFUEl2QTNDrjFx7Mkxq
+ObWp7uHSdOQHauEwQ/PR28CTjHisMtYsoC4aLP0E6tiAv+/97VRj0UA/T1LcYcQ9
+dRnQwgp7I0aeV2cOlNfPPTqlWLNhhKo=
 -----END CERTIFICATE-----
diff --git a/tests/testdata/certs/dns/cert-chain.pem b/tests/testdata/certs/dns/cert-chain.pem
index ae882d01b5..b6c4c45f23 100644
--- a/tests/testdata/certs/dns/cert-chain.pem
+++ b/tests/testdata/certs/dns/cert-chain.pem
@@ -1,19 +1,19 @@
 -----BEGIN CERTIFICATE-----
-MIIDGDCCAgCgAwIBAgIUFW8Q9nJKg/rDzm+EhcqzeULmJs8wDQYJKoZIhvcNAQEL
-BQAwGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDAgFw0yMDAzMTQwMDA0NDdaGA8y
-MjkzMTIyODAwMDQ0N1owGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDCCASIwDQYJ
-KoZIhvcNAQEBBQADggEPADCCAQoCggEBAL+oQ9HDmyelKSlWkNMSCYl+wrBxb7go
-0iGGalBrxkm0ScWSuGYwjivZfQ1nMvnrZGQZIsw2hDi5b7tgW1QS9G0jAzEPHN/f
-Bunzj3vKRp/U0WflAY8s7mR+bBa9yD9I1JI5kV7ScIt0ICK/dD7LrpT/ikO2xBlS
-hDFWMPV2fGTbneZSX9wDpaGSl4HwwxO9BpZTvE4Lft/gRj8d4eMc6gLETTSAzn24
-MiMy0AUp9r3bX6I/RPrrZDKEhnPG2dW7p7AgOBe7CWa2Zzvi6gnjOc4bqjSLWEgF
-bIGLRdrlYKQn+tzlHjm7+GYoMtGqkRx/pQFAV1SRVcvXjEMOcfHyw7sCAwEAAaNY
+MIIDGDCCAgCgAwIBAgIUFad49XzcT8c0W+ltJRmkcmncz5cwDQYJKoZIhvcNAQEL
+BQAwGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDAgFw0yMjAzMDMyMjM2NTJaGA8y
+Mjk1MTIxNzIyMzY1MlowGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDCCASIwDQYJ
+KoZIhvcNAQEBBQADggEPADCCAQoCggEBAPj9HqQOfJ5u5EajmNGV0DiDBc2DAkme
+88D0pey6lZtMCTluwOF3MihU40Q03Vyr1W89hNYuvl6tNnlvCuKmq+MWYn8F/4jr
+rVgt9MuKj7NvKBnjyZpgyYOZ5ZfD29/QEvmOz5xLb4lzhGsA+QqWiQvHjtYfkZoG
+UfbUsM0IFmY8DLpgVs9autzVrLCxzVlRHqUJ8Rw4tBuWnodYVTHoKNIQ9W8rNYIU
+4Yaia856mWtYRqBjXuGuI42h5fUt885ZVhkUoefPiKqErNtb1+8U6oH1KUfild9E
+P7ilnGNE/ipp81mtYSeTkdZP6nAQggJrYdE2t31mJdinPeQdl1umlKcCAwEAAaNY
 MFYwCQYDVR0TBAIwADALBgNVHQ8EBAMCBeAwHQYDVR0lBBYwFAYIKwYBBQUHAwIG
 CCsGAQUFBwMBMB0GA1UdEQQWMBSCEnNlcnZlci5kZWZhdWx0LnN2YzANBgkqhkiG
-9w0BAQsFAAOCAQEAtztb7XWNXrnq9vYve79kFt6ca8PWK47aZyegh9Elwk6xhcn2
-uCdEIbDdihmNAB66f/QCVPpVJ7C13+bXB2P6DeZ0OPTGyRhjdN6gFPBS3t1qz5vY
-IkN3t5x3RewHB0D1ZPW3QfBg1CtZHohzH+aNtlJLcMuadLxn4uyrsCJPGF37BNiF
-c0Etw3eUBsGqZ88wLLXe5cW536Knz2lLyjo02WueIAaQ/u8f/qRsAyr0uIatYbIT
-5ZtVkfJas3EUb678+oFae0nzwIv20943AAVa5ovPV58lfShnkPTessSH7Q7R1r9b
-0V0pMmlCRCQYVXEjGJk3GUWYYkXbKOFy8fJPqQ==
+9w0BAQsFAAOCAQEAH0mYLLKhppFbAp6BUPzux+iPNUYxvlpwQTjHVI/JuDnsQdHP
+QYbLDpU/Wg3lLFG7oz35uq5HQMqyGrFt30pVLo8O2B10EZMtb4xC+oU9qPuwfRG7
+4DZ4zI6m8iopS08fcnlA8HxLpjIEkZ1bOHQHWBjMppsc+i7uTxVH0VLZM2y7a6sK
+pqPHejCVymNb+MIKuP67FRqaL3FodRhsj3RR0ewQ0PplVYyk5lMOBd7RfEsHhuM6
+8khcoGxemZbN8lvSBT2a+M1PKypCQRrfwg8JIrn/FViBFjSSRewmHAPx/BaVji6p
+KG2uo2Trm+saCwtp+3lQ0ke4DRKUMfXItcz8EA==
 -----END CERTIFICATE-----
diff --git a/tests/testdata/certs/dns/key.pem b/tests/testdata/certs/dns/key.pem
index 237be66ede..cbaaea06cd 100644
--- a/tests/testdata/certs/dns/key.pem
+++ b/tests/testdata/certs/dns/key.pem
@@ -1,27 +1,27 @@
 -----BEGIN RSA PRIVATE KEY-----
-MIIEowIBAAKCAQEAv6hD0cObJ6UpKVaQ0xIJiX7CsHFvuCjSIYZqUGvGSbRJxZK4
-ZjCOK9l9DWcy+etkZBkizDaEOLlvu2BbVBL0bSMDMQ8c398G6fOPe8pGn9TRZ+UB
-jyzuZH5sFr3IP0jUkjmRXtJwi3QgIr90PsuulP+KQ7bEGVKEMVYw9XZ8ZNud5lJf
-3AOloZKXgfDDE70GllO8Tgt+3+BGPx3h4xzqAsRNNIDOfbgyIzLQBSn2vdtfoj9E
-+utkMoSGc8bZ1bunsCA4F7sJZrZnO+LqCeM5zhuqNItYSAVsgYtF2uVgpCf63OUe
-Obv4Zigy0aqRHH+lAUBXVJFVy9eMQw5x8fLDuwIDAQABAoIBAQCp/8OgRWPTR5F6
-JFmzjvU8q15fiU7yR+sTZ5nXmbE5YRxqjljX88u3o2Njxx0VRzE2BIG0J8siUYGE
-79Qyf9mlEIBdPK18+eO1EqA7ylDYprnT8PJwIAeYxfZ0OYRsddpvlXaXaAwB9lAm
-4jFnHd3ONdhbdbr9Rw8QgAZPQ/jUr9umqKAIhNGfN8ueSKgn/kKEwQO8DEJ4K31H
-pbksqErXtOJlOqM52o7J/290FtUwnw/U9xlDhmlbXQZnAp9LgBMh3L4eqKeMdpIZ
-gw1ublCs0oKWesI6g0JGBcda5ak2mVoF3LykEY3qPWaVHAFv+1RMZDFTwYhfonK5
-FmeuWo8hAoGBAN/UnNqI6P3LuzeEQu8uc9ubDtfewZ3wdhyVhvxyMDNrF5SIku6T
-f1gX0FzyN38zX+zbxg0oNc//JQ2aIjVdlcKzoyEt1+uxkU9H89138viC/3It3X0L
-XIYeyS0kgwlR7d0NVW4oY5dCJxdw9mCto8GSwdC6Dkm36m3j+YeXbT3rAoGBANsz
-51oHn3LVuZesPettS/mtDfBgr3o+kvL+FOU+4/grj7wcSe5AbsdF2Dp4FIisn8cw
-JyWfpwv22oVewhJv7ue/bgqpDOK3sbb3cofU5CuhWZWrZim+PaUsINSWG7h0qIAi
-cDYVAiS2Q3Og6wzGcHNwSfHykmO9dR79cGXG4I1xAoGAZXBQycWEegjLxwQ85XLS
-9PKXzoNYkgB5NskZTO1fJN5i6GDg3higfMcGGil3t0EMNyb+4dWYVbRXEYwJ9s97
-4KpSwuWo1ls59IkyXutOhQuxQwjAOTxd6Taoi7nhXObxet3hGrRZNiK33qrGLdhj
-v4WIKtU3RneAWd8YpjedPZMCgYBspX4+9lZGkb5Kefm6zdKBbw/gm9t96LPnMp5i
-vdg6ckTsxMy0BUWjCXBSEqCC2aTcEnbkxAwE9xZ6Umw4vTgZr504NXAIk2nHOUCQ
-KKnrIi8H6wtZF0zwqD2hgME+nLvDvs4FUEZeseQe1JyMGXM1FGviOwvp2cbKVG24
-VGmKUQKBgGsiHIZX9v/79xwJpa6zVduPAoa1MfI6gkRHxb2d2dzzw4fFFeesmDgX
-Rn3Y9F6oi6vQLzAKkXQCqVSzIPPatWaCVqV/1h4S0VSv1Qzj09bGQTBNl8mFoLO+
-kLkMaVcELCYfw5BPVJiB5BkVRfarMFyBqD3zb/ScC0uhqdddB45A
+MIIEoAIBAAKCAQEA+P0epA58nm7kRqOY0ZXQOIMFzYMCSZ7zwPSl7LqVm0wJOW7A
+4XcyKFTjRDTdXKvVbz2E1i6+Xq02eW8K4qar4xZifwX/iOutWC30y4qPs28oGePJ
+mmDJg5nll8Pb39AS+Y7PnEtviXOEawD5CpaJC8eO1h+RmgZR9tSwzQgWZjwMumBW
+z1q63NWssLHNWVEepQnxHDi0G5aeh1hVMego0hD1bys1ghThhqJrznqZa1hGoGNe
+4a4jjaHl9S3zzllWGRSh58+IqoSs21vX7xTqgfUpR+KV30Q/uKWcY0T+KmnzWa1h
+J5OR1k/qcBCCAmth0Ta3fWYl2Kc95B2XW6aUpwIDAQABAoIBABLg1jboJki47bPx
+aX/SlRQD0yfO3BiK5b5mjkIRr5AzxqtFG3i82d+29/hO7c299ZPWnUDEEIkLJws0
+ASZILtYr91HcOZC/FciOy3W3TsfzEJ5N6BY9amoA/ChyoDQrCZVEX7dg/Nb2HD8k
+tr03xrJO8iSIBjkga4yEeddB2rjrhF6HOzCZ+DZeld4r+mNCBeTWioCEgQX5od+0
+dOpzwDcQeB0T5gXxgk6HyuHH5tRj4a/Foq17f+1j4FzER+crYTskZ7FPQAxSKoaq
+5B77ahd0oZpW/jDDh6/S0W7i5dElK+kn2PYrszlW7qT9N6G/3J5dKhu1b1YtClnJ
+OmbXA+kCgYEA/TXrykXHne7HlbpB3M8ZwLHVqTLI1Rhd/a4WKyq0LJXOwGzrOHgV
+3vdv3pBL1Ij6IqO8Rep4xfy3mxZazE+RIxjbd9hXn2G7IaJVoCSu43xgqU+Py+8y
+gSHahqoV2ogxdsTWODZv8d+DYBxrH9RiEjthKVVSdSMmqmKGYCXRm1sCgYEA+7tK
+4mCbxyHyDQgq+lECMO3gFKWjGuetEeqafO115VGamvAC7sa8Yet2sNAzKxVwOE2t
+GiUABZOatSxhQwjxSoSyz2akZ9aOxJ+7hpICOwL3hGb013LJ2gfaN+Ve6blAG8+i
+9veYtUqCjyxyZhjWIhT+zEumGysM1iE4NEUpyaUCgYBiEBDqPE4Lnx8SoJjDUD2U
+AAtURMO8q3wzM8mIHDi9D6OLqGMMco1d1YwwsMd9XzMoWSsgFKeDXdUpreWqPKJj
+6kuMxW9h3XKgmwL4S1yj41Z4G8T/4Po5QogkIRjnklk9uwuPp52BGeQCQ99E63az
+wuVlIZMWpG9kC1YpRFxCAwJ/chBr3ifoblLzWvRYhRQv6fvfjXuAbLCNu5CScAkp
+Yr+XTp1ytr9WnQgUwAmXJe6Q6vJHfZFGcUBw2adCax2iOCHo5BGebW7LQk0eq9t9
+J0Aw582/I+Dflev49Vr5pu5KHk/3h/CpQXyBrF53d8vJO6hHzEp6+dSH/H8Q+J5e
+UQKBgB6BDqjKH41F7z7D2cVLzPRZXkf3zmS9Cs0JjuMtwyhAq5T5Q6XOaYjyXvY4
+Zae4Mzw31Hkrhjh5o2OrtiGKTdD4YlJRP/9CZflUImGUrfTagGciqLx9iSM5P4Qf
+howi98AVx34su6vFj+mfIYIYpEjAJg5ViJs4jPv9BJ3wuLo4
 -----END RSA PRIVATE KEY-----
diff --git a/tests/testdata/certs/dns/root-cert.pem b/tests/testdata/certs/dns/root-cert.pem
index 71062b7f8c..6f9c872974 100644
--- a/tests/testdata/certs/dns/root-cert.pem
+++ b/tests/testdata/certs/dns/root-cert.pem
@@ -1,19 +1,19 @@
 -----BEGIN CERTIFICATE-----
-MIIDEzCCAfugAwIBAgIUHu+RNpObSStNAQtgGqFPV70CNVEwDQYJKoZIhvcNAQEL
-BQAwGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDAgFw0yMDAzMTMwMDUwNTBaGA8y
-MjkzMTIyNzAwNTA1MFowGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDCCASIwDQYJ
-KoZIhvcNAQEBBQADggEPADCCAQoCggEBAN77xnr0H71SuBEGBI0amTmlvjwcgqZF
-mAw5c2pOCs3eUiJSz3tQYTRfVzaWBOrwEmn7RrQH8LJ5EVQm7Prg+aCT7koeBQyD
-G/Oqk2hpX1ArCLnjwEildUOlDSg+KBNJiImRFhMb2vnlFPfzkO5TsUk01OOnQTYm
-5XMzhQ0CuqX3stcAACsuKGxk3BX3r5zHmcFH4nBpbh9pb0DY2QZ+N+B6Qxe6o89q
-bT8gtWYLN1b3yGhn2WGeFkQtaud36Wj2jRSxWo2PBe8xc8qDBFJOt0zHg+u0Pt7t
-3mCrBIZ45FWEqX06FOupSe7HF2d7U/9pJGSZLe9iAJ/59WROlniJ7PUCAwEAAaNT
-MFEwHQYDVR0OBBYEFC1YD3F9RMLaW1xMuuQN27SmXzsMMB8GA1UdIwQYMBaAFC1Y
-D3F9RMLaW1xMuuQN27SmXzsMMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQEL
-BQADggEBAGqzLzTnusoOPF+tQ4jbhcXDCMvpDCCiAQVkkaorNGMWg2ny39vm3ymc
-HBhqWNNbeyZTj8hl7ffCoeUChZnIFkLO1ev3DVbLz179IP6dhqfDkuLhc/0g5Lfs
-FcNedAJh5TConI+x7dH1W2opPX4hXiiJBzcpPmfttD1EYES2VH85cPp95goUJwRp
-kMPAS5WiBLPv0c+0Msppe4iqxwuZZtJKRoHRDbvrHonMT1FmdOoEJ8xjeTxB8pjc
-tp5dA30dGNAxB7tOrzqmIidjXjYaxZGXKoC6BViw9ISfX9WNqVdvZVMJ7XwL6gu5
-Dk0yYXcMuGwMkMz+vJpypIBZAaCFES8=
+MIIDEzCCAfugAwIBAgIUf7oS8NPqTONgWOz1s8w4I6QWrv8wDQYJKoZIhvcNAQEL
+BQAwGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDAgFw0yMjAzMDMyMjM2NTFaGA8y
+Mjk1MTIxNzIyMzY1MVowGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDCCASIwDQYJ
+KoZIhvcNAQEBBQADggEPADCCAQoCggEBALD7cm5AqNsXTiAem/Wx2H26dYreh5OV
+g5DtTkAbN7lfoT98t9MFP5cgmzHcAKfaQ0RST+7tkp8j0IHgvZtvJYHAYCNjqXbo
+uKqgH2kdPDAjryWtUIk95md9ev91W50LKEEbMzxmBTTdsNoZ7HDuCYVSfenogxzV
+Uji4LheGcSvnDZ5QNvoog79Smn7m0qgEWXZmeD4pbQ5phRTE2Jz3VzTszr5Mzi7m
+9zPEvR8MLQcggVnWmHdCawVPGZrBbWfAtJXAFPNwNT9I9OXD2QubmEjv4IgTnxvw
+7fo+13CXKD2p8FYYpyZRaBwT/w9EcyEq5TA+HslWm5uwqi6Ezn9kgGcCAwEAAaNT
+MFEwHQYDVR0OBBYEFNPm0Yesg8sMVK3OhDMCIGW5FumoMB8GA1UdIwQYMBaAFNPm
+0Yesg8sMVK3OhDMCIGW5FumoMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQEL
+BQADggEBAFXlSpQZKB3lrcE+hbBfP4ToamIMv6Jn41tlPm0saV1Wqr+p3pAidRlu
+tEZEVC20V5IUVT/k3BGZLtPmrQQHfHHVNeQNwfPQpa/DeVWdSoXSfPxFiv4nIDEy
+orYGse/TDWGloID5NkDIz/WKSBIF0lurKnScC0sBJxyyW3UvTf4Fkf+fIS3ovqFN
+2k0zs9DRPu1NReuRc+BxHqCNq1z+mjmK7Z2aSDl7rSox1tFUEl2QTNDrjFx7Mkxq
+ObWp7uHSdOQHauEwQ/PR28CTjHisMtYsoC4aLP0E6tiAv+/97VRj0UA/T1LcYcQ9
+dRnQwgp7I0aeV2cOlNfPPTqlWLNhhKo=
 -----END CERTIFICATE-----
diff --git a/tests/testdata/certs/generate.sh b/tests/testdata/certs/generate.sh
index 245a3352de..8678dea811 100755
--- a/tests/testdata/certs/generate.sh
+++ b/tests/testdata/certs/generate.sh
@@ -110,27 +110,27 @@ cp "${WD}/pilot/root-cert.pem" "${WD}/mountedcerts-client/root-cert.pem"
 
 # Create a server certificate
 openssl genrsa -out "${WD}/pilot/key.pem" 2048
-openssl req -new -key "${WD}/pilot/key.pem" -out "${WD}/server.csr" -subj "/CN=cluster.local" -config "${WD}/server.conf"
+openssl req -new -sha256 -key "${WD}/pilot/key.pem" -out "${WD}/server.csr" -subj "/CN=cluster.local" -config "${WD}/server.conf"
 openssl x509 -req -in "${WD}/server.csr" -CA "${WD}/pilot/root-cert.pem" -CAkey "${WD}/pilot/ca-key.pem" -CAcreateserial -out "${WD}/pilot/cert-chain.pem"  -days 100000 -extensions v3_req -extfile "${WD}/server.conf"
 
 # Create a client certificate
 openssl genrsa -out "${WD}/default/key.pem" 2048
-openssl req -new -key "${WD}/default/key.pem" -out "${WD}/client.csr" -subj "/CN=cluster.local" -config "${WD}/client.conf"
+openssl req -new -sha256 -key "${WD}/default/key.pem" -out "${WD}/client.csr" -subj "/CN=cluster.local" -config "${WD}/client.conf"
 openssl x509 -req -in "${WD}/client.csr" -CA "${WD}/pilot/root-cert.pem" -CAkey "${WD}/pilot/ca-key.pem" -CAcreateserial -out "${WD}/default/cert-chain.pem" -days 100000 -extensions v3_req -extfile "${WD}/client.conf"
 
 # Create a DNS client certificate
 openssl genrsa -out "${WD}/dns/key.pem" 2048
-openssl req -new -key "${WD}/dns/key.pem" -out "${WD}/dns-client.csr" -subj "/CN=cluster.local" -config "${WD}/dns-client.conf"
+openssl req -new -sha256 -key "${WD}/dns/key.pem" -out "${WD}/dns-client.csr" -subj "/CN=cluster.local" -config "${WD}/dns-client.conf"
 openssl x509 -req -in "${WD}/dns-client.csr" -CA "${WD}/pilot/root-cert.pem" -CAkey "${WD}/pilot/ca-key.pem" -CAcreateserial -out "${WD}/dns/cert-chain.pem" -days 100000 -extensions v3_req -extfile "${WD}/dns-client.conf"
 
 # Create a server certificate for MountedCerts test
 openssl genrsa -out "${WD}/mountedcerts-server/key.pem" 2048
-openssl req -new -key "${WD}/mountedcerts-server/key.pem" -out "${WD}/mountedcerts-server.csr" -subj "/CN=cluster.local" -config "${WD}/mountedcerts-server.conf"
+openssl req -new -sha256 -key "${WD}/mountedcerts-server/key.pem" -out "${WD}/mountedcerts-server.csr" -subj "/CN=cluster.local" -config "${WD}/mountedcerts-server.conf"
 openssl x509 -req -in "${WD}/mountedcerts-server.csr" -CA "${WD}/pilot/root-cert.pem" -CAkey "${WD}/pilot/ca-key.pem" -CAcreateserial -out "${WD}/mountedcerts-server/cert-chain.pem" -days 100000 -extensions v3_req -extfile "${WD}/mountedcerts-server.conf"
 
 # Create a client certificate for MountedCerts test
 openssl genrsa -out "${WD}/mountedcerts-client/key.pem" 2048
-openssl req -new -key "${WD}/mountedcerts-client/key.pem" -out "${WD}/mountedcerts-client.csr" -subj "/CN=cluster.local" -config "${WD}/mountedcerts-client.conf"
+openssl req -new -sha256 -key "${WD}/mountedcerts-client/key.pem" -out "${WD}/mountedcerts-client.csr" -subj "/CN=cluster.local" -config "${WD}/mountedcerts-client.conf"
 openssl x509 -req -in "${WD}/mountedcerts-client.csr" -CA "${WD}/pilot/root-cert.pem" -CAkey "${WD}/pilot/ca-key.pem" -CAcreateserial -out "${WD}/mountedcerts-client/cert-chain.pem" -days 100000 -extensions v3_req -extfile "${WD}/mountedcerts-client.conf"
 
 
diff --git a/tests/testdata/certs/mountedcerts-client/cert-chain.pem b/tests/testdata/certs/mountedcerts-client/cert-chain.pem
index cd860da069..7beba5f0c8 100644
--- a/tests/testdata/certs/mountedcerts-client/cert-chain.pem
+++ b/tests/testdata/certs/mountedcerts-client/cert-chain.pem
@@ -1,20 +1,20 @@
 -----BEGIN CERTIFICATE-----
-MIIDSDCCAjCgAwIBAgIJALPneXFbvRa7MA0GCSqGSIb3DQEBBQUAMBgxFjAUBgNV
-BAMMDWNsdXN0ZXIubG9jYWwwIBcNMjAxMDIzMjM0NjA5WhgPMjI5NDA4MDgyMzQ2
-MDlaMBgxFjAUBgNVBAMMDWNsdXN0ZXIubG9jYWwwggEiMA0GCSqGSIb3DQEBAQUA
-A4IBDwAwggEKAoIBAQDm0iymZAZaZsqfjIUQhSacd2371lZ/H56x8OU7VozXkxiJ
-CN5JRSqygK5LMaO2DGm+mDoKCFMlyeiZLcBtXbouwhraFKASkK9Db5e0/RdPwhh7
-8hGL5mVtLlL7uPwuHydxibF8gnnICqgZhBzdvcqfYc374650UKhR2dT7SYGKyQfT
-JnJYAqf3AiXk1kUtjxNYJ44ip58DNI4+zcstojfgSzZUAFPZ9BTIPrYCy/U4K0zS
-501Wkr9szp/3THNP7xxUVk4lxMROjdwO8n8aS8JvlNvnisroMlmbOHuVmx+7TBv7
-YHgK4nM+tRnvdYvg5+iGWSZIGwjqlcOc4cCwPrkTAgMBAAGjgZIwgY8wCQYDVR0T
-BAIwADALBgNVHQ8EBAMCBeAwHQYDVR0lBBYwFAYIKwYBBQUHAwIGCCsGAQUFBwMB
-MFYGA1UdEQRPME2GMXNwaWZmZTovL2NsdXN0ZXIubG9jYWwvbnMvbW91bnRlZC1j
-ZXJ0cy9zYS9jbGllbnSCGGNsaWVudC5tb3VudGVkLWNlcnRzLnN2YzANBgkqhkiG
-9w0BAQUFAAOCAQEAyFG6EOGSPwHXTM9RiIqLm61WeBwxYhUtRHEPU/ganVGxfRmB
-rXwZOVNbQjVCSfrBH9ugvbI7Qpj+o5tmxuB5fNVLnQq+TjE9t3hO/VU9KNh8qARn
-LcNGZRmeeN9k8pX119sM2zWxN1LEKsbbc85Av5fOAxaADQy9Jx0sz+KPgruDGIft
-LGhLpgBb3yf1cdKVi3I5fWwW9UovADEk87+lDF67c4UfQ1/zzsnTRR7Uon2WOin2
-LpwEhb3+8hOKzTDzgqU1V3BHOE9eb0R15fConQ3KJi4sbpwI8IKr0Yxioi9M/rTx
-h1S+qxeHLC14uuvfMpUFouxJj/E7PTCGlKQw5g==
+MIIDUzCCAjugAwIBAgIUFad49XzcT8c0W+ltJRmkcmncz5kwDQYJKoZIhvcNAQEL
+BQAwGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDAgFw0yMjAzMDMyMjM2NTJaGA8y
+Mjk1MTIxNzIyMzY1MlowGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDCCASIwDQYJ
+KoZIhvcNAQEBBQADggEPADCCAQoCggEBANdt74Ga7JjLaO2rARTu2HIuMkYu5TRn
+ViDpxA5/7id904cs8Khyj9OWQPZt4zSgQq0t4jz8wtCgZ94S2SX3MHRJK4hPPaEh
+iXpQIvg5xg2wB+BKq7kvJQ1AWtaqiPgyeFjFRo+Hbpyrrx2JjSXW/eYc+STCSDqJ
+sXVZObGyskMQxVoBs7yOMj04blLYy2a9k3Y+R8tv9gC8fWaaKW+C1ZVE5W6gmw4R
+pJktVUdMhXBG+iNN2ofyF14eWffuCGs/IvXFh1ztyl8KEDn2PQvFD1Ip7fHtWVEu
+pDxVFGnsDl/uV5vuCnwt4DlJDq7WJDzBCKqEGpfR0U1LzrccuH3dIMkCAwEAAaOB
+kjCBjzAJBgNVHRMEAjAAMAsGA1UdDwQEAwIF4DAdBgNVHSUEFjAUBggrBgEFBQcD
+AgYIKwYBBQUHAwEwVgYDVR0RBE8wTYYxc3BpZmZlOi8vY2x1c3Rlci5sb2NhbC9u
+cy9tb3VudGVkLWNlcnRzL3NhL2NsaWVudIIYY2xpZW50Lm1vdW50ZWQtY2VydHMu
+c3ZjMA0GCSqGSIb3DQEBCwUAA4IBAQBNNZse8heVguDG+WUkwas54Ed94Mp5hboO
+k4oyXwv6H95ZOr+GeBLqNONt4l4r1Wjz9ytks5qavGU2rJCekX0pWI3Bu5E8PFtf
+BkjmEUObr4OdW2cq+c2Gs7dUzDWzhKu9VZ2oCw/A2yzHIHy05w/bFofx6+TOu7XP
+X2wW0cHBtP2QXaOc+AJ6huQgiXEs5B9ysGjpppTMbwukZ5bb9CVWPwN1XIfb+n5U
+PfZkcijvmWE9CvfZoqYiXROy0KGo1CDxY5IgL0F6L0tzVRlWHAUHpxItLfSTiEWf
+rJ2eLbVcm8rHPz9hC8k0bYP6FaGrfcofJrDyVC0AJDVfNAtstDiv
 -----END CERTIFICATE-----
diff --git a/tests/testdata/certs/mountedcerts-client/key.pem b/tests/testdata/certs/mountedcerts-client/key.pem
index 73070a4a40..1c8ec9d139 100644
--- a/tests/testdata/certs/mountedcerts-client/key.pem
+++ b/tests/testdata/certs/mountedcerts-client/key.pem
@@ -1,27 +1,27 @@
 -----BEGIN RSA PRIVATE KEY-----
-MIIEpQIBAAKCAQEA5tIspmQGWmbKn4yFEIUmnHdt+9ZWfx+esfDlO1aM15MYiQje
-SUUqsoCuSzGjtgxpvpg6CghTJcnomS3AbV26LsIa2hSgEpCvQ2+XtP0XT8IYe/IR
-i+ZlbS5S+7j8Lh8ncYmxfIJ5yAqoGYQc3b3Kn2HN++OudFCoUdnU+0mBiskH0yZy
-WAKn9wIl5NZFLY8TWCeOIqefAzSOPs3LLaI34Es2VABT2fQUyD62Asv1OCtM0udN
-VpK/bM6f90xzT+8cVFZOJcTETo3cDvJ/GkvCb5Tb54rK6DJZmzh7lZsfu0wb+2B4
-CuJzPrUZ73WL4OfohlkmSBsI6pXDnOHAsD65EwIDAQABAoIBAQCRZDgNwxmhXfgp
-0biLdGOi0TOtGxC22HL8wLGT0+u5aoVc5AIogvHnKFkhMi5d4KnHDVoZsa62Md1a
-h1AbU5uQEmIZlj5SE2Lihms37Of/LFKB2DOnPMQMps5MkvP+FU+iG8p5lDIjiFrk
-X6xTHFaCWHgIPx1HVPsBO8YFVwLLKK2REi0zo95A7eixveaW1ZL/iybNKCnHEmuN
-xZIKfOyL2XU4SUF6g4Oq7GFPtJG3914CQ5SB41ue1f5J5zpFHbv3YGqQip1048sC
-T9Xv3pp4eQLPRk4WaBrwhwFhYLThlH6NtvZe5EjBmwQhp6RMxYDJK3kMxyPpoL2r
-4rktdh/pAoGBAPPGOTXPZtdWieJaEQA2ZnJN+zFryoBrtBM3uNo7gOxpOYLSBhbv
-9x0yiZlbvn/bgwxAJYLk5kco5p/Mg7bBL8e/0WuOPZMKmoHKrOoCWj9H38XxSPrz
-4RfHBJL5/OolmKO2afZbOISiaFmAg94jaqoMh3MRetKu7f67XAjrfejfAoGBAPJl
-pUFPUdbPlkZVue/4gN6suss1fYgR6PuRsEM7WackTiO+BDlBsdFDfS+ti9JeAjQl
-4BeeGF1jHsuivaW6ZiXFc7OPjU6Z41dZ8+7nbsW1Q4SOUiaYjdlQE3nS3aEiRaX8
-HKNykERD1QEY8DDMSw8RdES1hh5HSP3ABrAfABJNAoGBAJ//LDF7LN6M9/bOcnYX
-PmorcIDhZImhJxGt8u+ZOmbwMUUBn+mSGjjZoKI9Ku00H/D8cSJWJiXyXFvpmVFr
-vnZ2GXdkZlmKDhq98Xuti5oZDHAYi3PaeZRnlUL6PNtZTbO12BpPiOrUxXRo+yPj
-TvDJRGVrxZdeJOqaAjAAkCU1AoGAV6o3SBV9zaip+qoKYyWGxgHGVLP1wH5puoII
-M2iFlhNlRzvznxmJS9uSrOmhBeYF+gIwt5nvyeaNPqyO1h9BzitJGwmx21mAhRj9
-RcvhnhNeZdRrFZZaxttVD9wZV6tjcRJq8Rn/3xCBOrJgO+XGx55SZH8aEF1LoqoO
-QSgVCIkCgYEA2AIoumrOsxJkDEgLix8N0CGgaKfN8etLFBfMk8VxEvPrpvq/6dZg
-aXLVnghNOkbql239IiN4JUAmYDRd13elH/Q9quW8Y49Qgo+9OVNwWN9WvQsrsa5P
-TsTg+hJ3ltvSiohBvkWu9B6qtNTHUCPRUdOAVJGI+vuZQ3zG7K127HI=
+MIIEpAIBAAKCAQEA123vgZrsmMto7asBFO7Yci4yRi7lNGdWIOnEDn/uJ33Thyzw
+qHKP05ZA9m3jNKBCrS3iPPzC0KBn3hLZJfcwdEkriE89oSGJelAi+DnGDbAH4Eqr
+uS8lDUBa1qqI+DJ4WMVGj4dunKuvHYmNJdb95hz5JMJIOomxdVk5sbKyQxDFWgGz
+vI4yPThuUtjLZr2Tdj5Hy2/2ALx9Zpopb4LVlUTlbqCbDhGkmS1VR0yFcEb6I03a
+h/IXXh5Z9+4Iaz8i9cWHXO3KXwoQOfY9C8UPUint8e1ZUS6kPFUUaewOX+5Xm+4K
+fC3gOUkOrtYkPMEIqoQal9HRTUvOtxy4fd0gyQIDAQABAoIBAErCbOZGNCsGE+C+
+gj6YcGaXX3z5kzLvVXVQimgZvzTXNovnbZCXSRA9SUf7dGhz4Q+mJ6fQc27g64qm
+cdL6CEeiiMfh79o2ADaBDtGjgvaHCwSsBRHbqXkCx0NNpQC0ywzLUSRH9HvMHJYc
+gO4LwMqVbWRzQ4uE8x9tGldvUutwiHsRlAzUo0KH7xs/75GHalkqNQ1O7aSblGPi
+/2zWrJ2t/KEF2iieMOe/YUcjT/Fhj8uP/kSeTWXTmc4FgkZRtsvrgU7Bye2RBmCv
+nBgspGrBYVIU+CeMHuDGNMAelBboK0IaHSQoEBwi5VqQuZotdwydiamo93AG21lu
+b2HRGk0CgYEA7YxqcSnm6cY05T86rizc8IRpGzXbulGRPRtA4FYrx5Wjo2WMMHq2
+6S0o9C5e0p35TtGcMlh1OfQ1L0afYfKRB/vaS1HMMFn3briQ5aPIIcKP814gAkcJ
+Tek3RUbA7GyZOuRyUT1LGsWekpHM+o82TNixayK13MCOi8rllU02d5cCgYEA6Cmw
+Qj3osR+NBdxk6YKDlTG9PbL94m1/afLE77n80d0oF7ZGDd05mWUlyZU8RvfKypCh
+PG8lfndUBrAweBNDAzIsjUEMI7xNSaGoA7I5xbtj8roAnuHYc9byv4s6e9ZPoOgY
+PQ2GaT0bZs+9DsPw51uUxelPrSyYwNe7Q9SJNp8CgYEAiz2mWuTT3K347XONovDb
+JQSIU5HTVThgqO/jrFv2V4wIGkv4y6Xyt2VoPxE7NbuxmhjVtNETDcBuuGJuJEy5
+5HMlNscgqCrkFXuMO38wXVGdbWr72jnnY0rTwgDCLxNNEi88LxB9iJjT5VgfTbNK
+ger2vDThYx9fruNkhYp35csCgYEAvlVXNKeOuo+0TQbIEIy4lbsme4vgCx9cIx8u
+B5+CPwfCE9aSjlE9JJ5BIYfg2AnUTsfN+D7jDVCFKcqOohKgkOE0l0Wpzy5UarZG
+iWk3Zf1gTqHhWmCr1SR29UQO2XH07GT7YipRdLvtWBed1pobSYHScYL6mG4Eea0j
+0NNQ/t8CgYAFdgPS0FSHP7rB0sJ/QJ2bSfkfEdVRGctSHvxNYftH1pg5HHVYdvg0
+TzrgqqQpiDhiiOdJrqzoDuYP74bvslDX9UmX6LAaF8ZDDFakJLl1SnTh1XPx6ez2
+wnXq4r0XKt1WqjoW/J5k1r9vIMxif6fK1rbNJ03SY7SG+/sQc7FlFg==
 -----END RSA PRIVATE KEY-----
diff --git a/tests/testdata/certs/mountedcerts-client/root-cert.pem b/tests/testdata/certs/mountedcerts-client/root-cert.pem
index 71062b7f8c..6f9c872974 100644
--- a/tests/testdata/certs/mountedcerts-client/root-cert.pem
+++ b/tests/testdata/certs/mountedcerts-client/root-cert.pem
@@ -1,19 +1,19 @@
 -----BEGIN CERTIFICATE-----
-MIIDEzCCAfugAwIBAgIUHu+RNpObSStNAQtgGqFPV70CNVEwDQYJKoZIhvcNAQEL
-BQAwGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDAgFw0yMDAzMTMwMDUwNTBaGA8y
-MjkzMTIyNzAwNTA1MFowGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDCCASIwDQYJ
-KoZIhvcNAQEBBQADggEPADCCAQoCggEBAN77xnr0H71SuBEGBI0amTmlvjwcgqZF
-mAw5c2pOCs3eUiJSz3tQYTRfVzaWBOrwEmn7RrQH8LJ5EVQm7Prg+aCT7koeBQyD
-G/Oqk2hpX1ArCLnjwEildUOlDSg+KBNJiImRFhMb2vnlFPfzkO5TsUk01OOnQTYm
-5XMzhQ0CuqX3stcAACsuKGxk3BX3r5zHmcFH4nBpbh9pb0DY2QZ+N+B6Qxe6o89q
-bT8gtWYLN1b3yGhn2WGeFkQtaud36Wj2jRSxWo2PBe8xc8qDBFJOt0zHg+u0Pt7t
-3mCrBIZ45FWEqX06FOupSe7HF2d7U/9pJGSZLe9iAJ/59WROlniJ7PUCAwEAAaNT
-MFEwHQYDVR0OBBYEFC1YD3F9RMLaW1xMuuQN27SmXzsMMB8GA1UdIwQYMBaAFC1Y
-D3F9RMLaW1xMuuQN27SmXzsMMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQEL
-BQADggEBAGqzLzTnusoOPF+tQ4jbhcXDCMvpDCCiAQVkkaorNGMWg2ny39vm3ymc
-HBhqWNNbeyZTj8hl7ffCoeUChZnIFkLO1ev3DVbLz179IP6dhqfDkuLhc/0g5Lfs
-FcNedAJh5TConI+x7dH1W2opPX4hXiiJBzcpPmfttD1EYES2VH85cPp95goUJwRp
-kMPAS5WiBLPv0c+0Msppe4iqxwuZZtJKRoHRDbvrHonMT1FmdOoEJ8xjeTxB8pjc
-tp5dA30dGNAxB7tOrzqmIidjXjYaxZGXKoC6BViw9ISfX9WNqVdvZVMJ7XwL6gu5
-Dk0yYXcMuGwMkMz+vJpypIBZAaCFES8=
+MIIDEzCCAfugAwIBAgIUf7oS8NPqTONgWOz1s8w4I6QWrv8wDQYJKoZIhvcNAQEL
+BQAwGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDAgFw0yMjAzMDMyMjM2NTFaGA8y
+Mjk1MTIxNzIyMzY1MVowGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDCCASIwDQYJ
+KoZIhvcNAQEBBQADggEPADCCAQoCggEBALD7cm5AqNsXTiAem/Wx2H26dYreh5OV
+g5DtTkAbN7lfoT98t9MFP5cgmzHcAKfaQ0RST+7tkp8j0IHgvZtvJYHAYCNjqXbo
+uKqgH2kdPDAjryWtUIk95md9ev91W50LKEEbMzxmBTTdsNoZ7HDuCYVSfenogxzV
+Uji4LheGcSvnDZ5QNvoog79Smn7m0qgEWXZmeD4pbQ5phRTE2Jz3VzTszr5Mzi7m
+9zPEvR8MLQcggVnWmHdCawVPGZrBbWfAtJXAFPNwNT9I9OXD2QubmEjv4IgTnxvw
+7fo+13CXKD2p8FYYpyZRaBwT/w9EcyEq5TA+HslWm5uwqi6Ezn9kgGcCAwEAAaNT
+MFEwHQYDVR0OBBYEFNPm0Yesg8sMVK3OhDMCIGW5FumoMB8GA1UdIwQYMBaAFNPm
+0Yesg8sMVK3OhDMCIGW5FumoMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQEL
+BQADggEBAFXlSpQZKB3lrcE+hbBfP4ToamIMv6Jn41tlPm0saV1Wqr+p3pAidRlu
+tEZEVC20V5IUVT/k3BGZLtPmrQQHfHHVNeQNwfPQpa/DeVWdSoXSfPxFiv4nIDEy
+orYGse/TDWGloID5NkDIz/WKSBIF0lurKnScC0sBJxyyW3UvTf4Fkf+fIS3ovqFN
+2k0zs9DRPu1NReuRc+BxHqCNq1z+mjmK7Z2aSDl7rSox1tFUEl2QTNDrjFx7Mkxq
+ObWp7uHSdOQHauEwQ/PR28CTjHisMtYsoC4aLP0E6tiAv+/97VRj0UA/T1LcYcQ9
+dRnQwgp7I0aeV2cOlNfPPTqlWLNhhKo=
 -----END CERTIFICATE-----
diff --git a/tests/testdata/certs/mountedcerts-server/cert-chain.pem b/tests/testdata/certs/mountedcerts-server/cert-chain.pem
index 5e33300bc8..e9fee31659 100644
--- a/tests/testdata/certs/mountedcerts-server/cert-chain.pem
+++ b/tests/testdata/certs/mountedcerts-server/cert-chain.pem
@@ -1,20 +1,20 @@
 -----BEGIN CERTIFICATE-----
-MIIDSDCCAjCgAwIBAgIJALPneXFbvRa5MA0GCSqGSIb3DQEBBQUAMBgxFjAUBgNV
-BAMMDWNsdXN0ZXIubG9jYWwwIBcNMjAxMDIzMjMzMzQ1WhgPMjI5NDA4MDgyMzMz
-NDVaMBgxFjAUBgNVBAMMDWNsdXN0ZXIubG9jYWwwggEiMA0GCSqGSIb3DQEBAQUA
-A4IBDwAwggEKAoIBAQD6XVhQ9pw3rttMJjhiQLqNWsVaexA2RN1cDA2DmzhSVdaH
-2XQybondfmNvEmv3eVPbCYRlydgYIdnQnL6bCXGMiodsarIwR+wRUWm1ltZpWu2+
-JOuiwRf+hProZdiN21v+eAo17h8ErnD442eYpdd8gRZhIder/GYWaVJknHwtbeM2
-qbbkgcACsIy0fv9xUTLfSOWqrsD+i1rVkJCgl452NIku32JrLQ08PSiUGwiYCyYz
-FpDYo/tWGhOavkShyte0QqbnLR7Kf7BH3olC02EsAqvDva8Zy7/T19IUslgr5hgH
-GacYtEIHVun7AcrdZob+JHFScHKECQ+/u1FJrGjXAgMBAAGjgZIwgY8wCQYDVR0T
-BAIwADALBgNVHQ8EBAMCBeAwHQYDVR0lBBYwFAYIKwYBBQUHAwIGCCsGAQUFBwMB
-MFYGA1UdEQRPME2GMXNwaWZmZTovL2NsdXN0ZXIubG9jYWwvbnMvbW91bnRlZC1j
-ZXJ0cy9zYS9zZXJ2ZXKCGHNlcnZlci5tb3VudGVkLWNlcnRzLnN2YzANBgkqhkiG
-9w0BAQUFAAOCAQEA3aGFvhZjU5NqlnY/2iSgQP6FHgzgfwMuRmifEWpaHawnM0cr
-0IAqeYjwMWZ9lVqGfimBVJ3QE7zimw4mqTL2tpFs/0qTQaHqHMJ3dstet/OGGFhQ
-sSEkE4oNoKuIoPbZHdUm3CR2m7jbL+tX77tC37UsFfgfus0L4ffaU/GiYHgk9AMa
-mHJbVN50Ltln3KBGJ72EScsrPRqupX9uNy9Irj8X19cDfVUx+BqFWFwthLcfrnF4
-FeQE16HlqOA6QGg11XR6NRvjhfhfBZts7DpEksKFU0r6zRfADjItye9m+PIk4C7e
-DC7V+j4bixLv3sMl7rIg6Wb/JMvU3xiN5IjuGg==
+MIIDUzCCAjugAwIBAgIUFad49XzcT8c0W+ltJRmkcmncz5gwDQYJKoZIhvcNAQEL
+BQAwGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDAgFw0yMjAzMDMyMjM2NTJaGA8y
+Mjk1MTIxNzIyMzY1MlowGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDCCASIwDQYJ
+KoZIhvcNAQEBBQADggEPADCCAQoCggEBAMBbFyebP2aVeF+hgPfa6SLK4sAL1Xuv
+DDK2Bh/IMklfJ/o0CjFd6sOPrYQ//dOx/VujEGxUlEIyD21fAgUfnySJWPDOImre
+WLnAIXw80+21Bl0ezlW7bwObtBhzVnGRO8A+wzE943n8E07OCxuCazmBsaAf4peu
+8xLbsrYUXBSYMxjFIZUBv9IHAhHs7DKJXwXoBfkylkWPrxmIOr/mBwxF8rg3uYT4
+MlEXBFFjlCcbncFw4TnehDbShi36SoAtO2yHyTdQdYjWizpNnCh0mzNgQXTcZm61
+rXFIDCjxdb6/aBf20VQynOgEMdc7gXRXbdVf0geinFSprJ2vVvQUqPkCAwEAAaOB
+kjCBjzAJBgNVHRMEAjAAMAsGA1UdDwQEAwIF4DAdBgNVHSUEFjAUBggrBgEFBQcD
+AgYIKwYBBQUHAwEwVgYDVR0RBE8wTYYxc3BpZmZlOi8vY2x1c3Rlci5sb2NhbC9u
+cy9tb3VudGVkLWNlcnRzL3NhL3NlcnZlcoIYc2VydmVyLm1vdW50ZWQtY2VydHMu
+c3ZjMA0GCSqGSIb3DQEBCwUAA4IBAQA0omHF0S5f9kMe73rFHbKRd6AnfUBUZBVu
+udpvObBc2sptS+ECp08q54BDcB2/d5IvD4fUMju6+BVOqD3LKEmkNJL5m2pGINPM
+9L5N8UrqYo4wHTWJRno7FqbxQz3Lby8V0sFRu3FLR6lU+m3dKrwZChxopBQI4tHE
+88uIMHMk+GtMeiSfhphZdlWOek1lbwgouRyhd5kPiUoCU5ydl13VX14JaFU/VVb2
+iETMSN5l8i4udoWVNdYa6VbPOuBUKLJIvY7xYLHU9Bc/ZjqHkBvsqQe2Qgm3jHt5
+vfJ3PLc1RS7ZXW3CKJM0TIw4Rja/dnRbfEzDGgH/NbhcnNfaxUQi
 -----END CERTIFICATE-----
diff --git a/tests/testdata/certs/mountedcerts-server/key.pem b/tests/testdata/certs/mountedcerts-server/key.pem
index f9b150d8cc..012d733336 100644
--- a/tests/testdata/certs/mountedcerts-server/key.pem
+++ b/tests/testdata/certs/mountedcerts-server/key.pem
@@ -1,27 +1,27 @@
 -----BEGIN RSA PRIVATE KEY-----
-MIIEpQIBAAKCAQEA+l1YUPacN67bTCY4YkC6jVrFWnsQNkTdXAwNg5s4UlXWh9l0
-Mm6J3X5jbxJr93lT2wmEZcnYGCHZ0Jy+mwlxjIqHbGqyMEfsEVFptZbWaVrtviTr
-osEX/oT66GXYjdtb/ngKNe4fBK5w+ONnmKXXfIEWYSHXq/xmFmlSZJx8LW3jNqm2
-5IHAArCMtH7/cVEy30jlqq7A/ota1ZCQoJeOdjSJLt9iay0NPD0olBsImAsmMxaQ
-2KP7VhoTmr5EocrXtEKm5y0eyn+wR96JQtNhLAKrw72vGcu/09fSFLJYK+YYBxmn
-GLRCB1bp+wHK3WaG/iRxUnByhAkPv7tRSaxo1wIDAQABAoIBAFB84+Cbd5tbJnsz
-qlrD7nV1Ig81kv+vYe6unUWbCB9II0DnQrjX2pCPik3yAS29Hp1DUQIcQtF3SHwN
-eCWFgrCo9GKBZqmo5ZansrFDPcj/B4FigzJ+m0Ui4TsHSvhH9io30yiFVgl+Pyl0
-iiCD6GPJaDVv2NMcwIEh+PAJcs5za5is15XsP9M1bb8K81p68oEmmeki1lDuDuZ4
-ketKYdRtoO8dr8Gv4Iui5D0zo15lREsk/QDaxg4lT3zlTPIfp93UXxUDVpxiwKK9
-OJ2UgDRrZUDg46xGeqvIwsRMbWNNrW2RNv6JOQpMBI4skPd/0wV2NRv/FdOcZjlR
-6ynacSECgYEA/woPxygucMefq2Lgta+xt/tlZ7R+HftgGa6IQgZcp69IUCvqzs6z
-QLklV5XG4I4s+tzGdBB87vimTZCbV9T/rgYOYCvTb3kRAmU9IAbOmtP23cxD5MZ/
-zUxplULOYZjvdv39APbpIeLbFt9cTYF83OD6vN38PLpnxXbSUfacTEcCgYEA+07G
-hpympBvicPgVpnvvlrzx4KbTXRa6D+Aj8d8CvXrIJK/QUibTkHhPaBvYCzFWV49n
-q6l/2HuwxhkMN7DaKB6LmngBwX+pKNrkcP0plNfym7iIva/SP4MAqD4A9U0xsjao
-EdmE8vj758kw+fAXXloL6r24K5jUpw7EnNvxlvECgYEA17weHpx5tELYaLRqWqX1
-Eilg3PmL2TbDUtYVQqx5Ags+gzqzFB86+BvlX5I9ucPqfoX0hUdRNyA0JNaW4RQ2
-93D+BhHsd5YUck9LBmVbrRJx96qLH+FdSD9D7NOnGl2w6pnswz0ZdeWc6FsmjBVl
-lr3ku5CXhpMzppBdVqHiEo8CgYEA+mUkM63T8AIEnlDhqY8lSskaK5+tS7xASV8o
-EXAF1+uFjeBLrf8Xn4PwgqfznDtdb0LopcbuJjIgEVGVtcuVopHEwXU/MqBTWOy4
-3sewbai5B2VXgcC0uSl7NycIZ2Gx2YLR9W8JI9rni6wQC4iokA8ifpLn9TCC+a0i
-Q7R2Q9ECgYEA4DTMh1RJ0s2m4qa+APxSzleMwyxSsDn0XtYQLkvlYU/yoqzBv40i
-6vlJzzULr6BHlBZMMNlWX5keA9TWqX7xl321b7fP4B7hFY/PnkMeeUN1gx7/vcF+
-2A7cLiXQyxech4+W3fbf6pV2Xxlh2H+SlxLw1pYz1LlFTAtPB04vzZw=
+MIIEpAIBAAKCAQEAwFsXJ5s/ZpV4X6GA99rpIsriwAvVe68MMrYGH8gySV8n+jQK
+MV3qw4+thD/907H9W6MQbFSUQjIPbV8CBR+fJIlY8M4iat5YucAhfDzT7bUGXR7O
+VbtvA5u0GHNWcZE7wD7DMT3jefwTTs4LG4JrOYGxoB/il67zEtuythRcFJgzGMUh
+lQG/0gcCEezsMolfBegF+TKWRY+vGYg6v+YHDEXyuDe5hPgyURcEUWOUJxudwXDh
+Od6ENtKGLfpKgC07bIfJN1B1iNaLOk2cKHSbM2BBdNxmbrWtcUgMKPF1vr9oF/bR
+VDKc6AQx1zuBdFdt1V/SB6KcVKmsna9W9BSo+QIDAQABAoIBAQChc9ObitNMZ2kB
+261EXJoSefiY+VXW2raCWgYJ5x9ONnB3sfZcOVgzjQU3W/7RETR3FgdS5isK5LnO
+M1CevEv3ftEITfzVdotCXeG9RdEcVDVjM9g8XXZwsOAZJbYI3L4qPoPXcPreCJas
+LoVkMbOOMS3GGP04E8Ays/Tsn2oqXZ+5pBnSpD+2JPAZa7rzK3//yCTyqPSH4YuL
+zJBcnzLq9Qqxyv1ZAdE6oMWPycoEDqIwEA8Aia3Bx8PMyOEs47VUEeOuH/S9LHln
+9s4G76jVhBQBHmJ4obC6bRSQbWD+DbtEX7j37/Lpw2cNe4VIlLA3jYTXRApwv8y5
+RaJ53QABAoGBAPERqNSZLo7ZYMv2R8SQz+eM/EW47i6TGO0SlCR/1eaImnybOLpI
+Kx5rGvshEQxfD5+jJ/A9SJhgtogJQRR7+q//BvAQUqg4Y6c4yn6hQuCAa0La6wVv
+6xwC3AiutB5G6KwR3241VW7I4Blgg+XKGWdYqzVhZoI9CJ9yyqCjW+r5AoGBAMxF
+C4faBzqta83edPUm1RPKMgVgOJtLmo7iWty5EMyfe0IbZ+r3G8+PFAMBfSbEgcMm
+iiygu9SNCVnvDUm40bvdwh2CHaiLHn6kgh6UwdBq93tgEyr6TeAUx65OAA2MeH8t
+uniauiHwbIsOdWYIigKO4n9KAnXhq8kkQVzQgC4BAoGBAJO/thEvHWjx9DCt7kic
+VMBfGZMGB/1FrKZIl8FTFF9p4/1xCTCq4LqQ/IVkO9WwQZwe3HCSrvatd9Iikd+j
+KP9LOA5UgsSVYwZveTRpU9rS+mqMUDUR5jBR1FgB82UHIr2rapxaVdVDio4jKFPj
+LPS5VJ2v56r1C17mdShFVyXJAoGATrgtAWWcc5hxexzHpdIZO08oh7nVFsPEBFxm
+97i6KKEtNwSov3wLU6mn5KCmQlJoT3SCu0AWIotXlzjwvgBjkyqarBSAqXszfUqY
+eoCJUHBofzs/J8jeSgS4DwwwKQiBR0GwIto98OZebhhcVb3ksaeu0a8L07HsKc+3
+ALW5MAECgYBm5lyP7oQrUi5O1FDSlWnbFIljp8Y9zxL8EMheZ3AGHl8HWeVldFZY
+0hCBeH7S9x/8FI1+i5yrcNGPrE5NmdJPymxEV/4rG4EovsiqNLbJcgCBLpDUpKN9
+yrSUfBmpJgLl7u6iApRIOBr8tF8VDbzPd3tkCxLouOAv6Gy4l4kZyA==
 -----END RSA PRIVATE KEY-----
diff --git a/tests/testdata/certs/mountedcerts-server/root-cert.pem b/tests/testdata/certs/mountedcerts-server/root-cert.pem
index 71062b7f8c..6f9c872974 100644
--- a/tests/testdata/certs/mountedcerts-server/root-cert.pem
+++ b/tests/testdata/certs/mountedcerts-server/root-cert.pem
@@ -1,19 +1,19 @@
 -----BEGIN CERTIFICATE-----
-MIIDEzCCAfugAwIBAgIUHu+RNpObSStNAQtgGqFPV70CNVEwDQYJKoZIhvcNAQEL
-BQAwGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDAgFw0yMDAzMTMwMDUwNTBaGA8y
-MjkzMTIyNzAwNTA1MFowGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDCCASIwDQYJ
-KoZIhvcNAQEBBQADggEPADCCAQoCggEBAN77xnr0H71SuBEGBI0amTmlvjwcgqZF
-mAw5c2pOCs3eUiJSz3tQYTRfVzaWBOrwEmn7RrQH8LJ5EVQm7Prg+aCT7koeBQyD
-G/Oqk2hpX1ArCLnjwEildUOlDSg+KBNJiImRFhMb2vnlFPfzkO5TsUk01OOnQTYm
-5XMzhQ0CuqX3stcAACsuKGxk3BX3r5zHmcFH4nBpbh9pb0DY2QZ+N+B6Qxe6o89q
-bT8gtWYLN1b3yGhn2WGeFkQtaud36Wj2jRSxWo2PBe8xc8qDBFJOt0zHg+u0Pt7t
-3mCrBIZ45FWEqX06FOupSe7HF2d7U/9pJGSZLe9iAJ/59WROlniJ7PUCAwEAAaNT
-MFEwHQYDVR0OBBYEFC1YD3F9RMLaW1xMuuQN27SmXzsMMB8GA1UdIwQYMBaAFC1Y
-D3F9RMLaW1xMuuQN27SmXzsMMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQEL
-BQADggEBAGqzLzTnusoOPF+tQ4jbhcXDCMvpDCCiAQVkkaorNGMWg2ny39vm3ymc
-HBhqWNNbeyZTj8hl7ffCoeUChZnIFkLO1ev3DVbLz179IP6dhqfDkuLhc/0g5Lfs
-FcNedAJh5TConI+x7dH1W2opPX4hXiiJBzcpPmfttD1EYES2VH85cPp95goUJwRp
-kMPAS5WiBLPv0c+0Msppe4iqxwuZZtJKRoHRDbvrHonMT1FmdOoEJ8xjeTxB8pjc
-tp5dA30dGNAxB7tOrzqmIidjXjYaxZGXKoC6BViw9ISfX9WNqVdvZVMJ7XwL6gu5
-Dk0yYXcMuGwMkMz+vJpypIBZAaCFES8=
+MIIDEzCCAfugAwIBAgIUf7oS8NPqTONgWOz1s8w4I6QWrv8wDQYJKoZIhvcNAQEL
+BQAwGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDAgFw0yMjAzMDMyMjM2NTFaGA8y
+Mjk1MTIxNzIyMzY1MVowGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDCCASIwDQYJ
+KoZIhvcNAQEBBQADggEPADCCAQoCggEBALD7cm5AqNsXTiAem/Wx2H26dYreh5OV
+g5DtTkAbN7lfoT98t9MFP5cgmzHcAKfaQ0RST+7tkp8j0IHgvZtvJYHAYCNjqXbo
+uKqgH2kdPDAjryWtUIk95md9ev91W50LKEEbMzxmBTTdsNoZ7HDuCYVSfenogxzV
+Uji4LheGcSvnDZ5QNvoog79Smn7m0qgEWXZmeD4pbQ5phRTE2Jz3VzTszr5Mzi7m
+9zPEvR8MLQcggVnWmHdCawVPGZrBbWfAtJXAFPNwNT9I9OXD2QubmEjv4IgTnxvw
+7fo+13CXKD2p8FYYpyZRaBwT/w9EcyEq5TA+HslWm5uwqi6Ezn9kgGcCAwEAAaNT
+MFEwHQYDVR0OBBYEFNPm0Yesg8sMVK3OhDMCIGW5FumoMB8GA1UdIwQYMBaAFNPm
+0Yesg8sMVK3OhDMCIGW5FumoMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQEL
+BQADggEBAFXlSpQZKB3lrcE+hbBfP4ToamIMv6Jn41tlPm0saV1Wqr+p3pAidRlu
+tEZEVC20V5IUVT/k3BGZLtPmrQQHfHHVNeQNwfPQpa/DeVWdSoXSfPxFiv4nIDEy
+orYGse/TDWGloID5NkDIz/WKSBIF0lurKnScC0sBJxyyW3UvTf4Fkf+fIS3ovqFN
+2k0zs9DRPu1NReuRc+BxHqCNq1z+mjmK7Z2aSDl7rSox1tFUEl2QTNDrjFx7Mkxq
+ObWp7uHSdOQHauEwQ/PR28CTjHisMtYsoC4aLP0E6tiAv+/97VRj0UA/T1LcYcQ9
+dRnQwgp7I0aeV2cOlNfPPTqlWLNhhKo=
 -----END CERTIFICATE-----
diff --git a/tests/testdata/certs/pilot/ca-cert.pem b/tests/testdata/certs/pilot/ca-cert.pem
index 71062b7f8c..6f9c872974 100644
--- a/tests/testdata/certs/pilot/ca-cert.pem
+++ b/tests/testdata/certs/pilot/ca-cert.pem
@@ -1,19 +1,19 @@
 -----BEGIN CERTIFICATE-----
-MIIDEzCCAfugAwIBAgIUHu+RNpObSStNAQtgGqFPV70CNVEwDQYJKoZIhvcNAQEL
-BQAwGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDAgFw0yMDAzMTMwMDUwNTBaGA8y
-MjkzMTIyNzAwNTA1MFowGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDCCASIwDQYJ
-KoZIhvcNAQEBBQADggEPADCCAQoCggEBAN77xnr0H71SuBEGBI0amTmlvjwcgqZF
-mAw5c2pOCs3eUiJSz3tQYTRfVzaWBOrwEmn7RrQH8LJ5EVQm7Prg+aCT7koeBQyD
-G/Oqk2hpX1ArCLnjwEildUOlDSg+KBNJiImRFhMb2vnlFPfzkO5TsUk01OOnQTYm
-5XMzhQ0CuqX3stcAACsuKGxk3BX3r5zHmcFH4nBpbh9pb0DY2QZ+N+B6Qxe6o89q
-bT8gtWYLN1b3yGhn2WGeFkQtaud36Wj2jRSxWo2PBe8xc8qDBFJOt0zHg+u0Pt7t
-3mCrBIZ45FWEqX06FOupSe7HF2d7U/9pJGSZLe9iAJ/59WROlniJ7PUCAwEAAaNT
-MFEwHQYDVR0OBBYEFC1YD3F9RMLaW1xMuuQN27SmXzsMMB8GA1UdIwQYMBaAFC1Y
-D3F9RMLaW1xMuuQN27SmXzsMMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQEL
-BQADggEBAGqzLzTnusoOPF+tQ4jbhcXDCMvpDCCiAQVkkaorNGMWg2ny39vm3ymc
-HBhqWNNbeyZTj8hl7ffCoeUChZnIFkLO1ev3DVbLz179IP6dhqfDkuLhc/0g5Lfs
-FcNedAJh5TConI+x7dH1W2opPX4hXiiJBzcpPmfttD1EYES2VH85cPp95goUJwRp
-kMPAS5WiBLPv0c+0Msppe4iqxwuZZtJKRoHRDbvrHonMT1FmdOoEJ8xjeTxB8pjc
-tp5dA30dGNAxB7tOrzqmIidjXjYaxZGXKoC6BViw9ISfX9WNqVdvZVMJ7XwL6gu5
-Dk0yYXcMuGwMkMz+vJpypIBZAaCFES8=
+MIIDEzCCAfugAwIBAgIUf7oS8NPqTONgWOz1s8w4I6QWrv8wDQYJKoZIhvcNAQEL
+BQAwGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDAgFw0yMjAzMDMyMjM2NTFaGA8y
+Mjk1MTIxNzIyMzY1MVowGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDCCASIwDQYJ
+KoZIhvcNAQEBBQADggEPADCCAQoCggEBALD7cm5AqNsXTiAem/Wx2H26dYreh5OV
+g5DtTkAbN7lfoT98t9MFP5cgmzHcAKfaQ0RST+7tkp8j0IHgvZtvJYHAYCNjqXbo
+uKqgH2kdPDAjryWtUIk95md9ev91W50LKEEbMzxmBTTdsNoZ7HDuCYVSfenogxzV
+Uji4LheGcSvnDZ5QNvoog79Smn7m0qgEWXZmeD4pbQ5phRTE2Jz3VzTszr5Mzi7m
+9zPEvR8MLQcggVnWmHdCawVPGZrBbWfAtJXAFPNwNT9I9OXD2QubmEjv4IgTnxvw
+7fo+13CXKD2p8FYYpyZRaBwT/w9EcyEq5TA+HslWm5uwqi6Ezn9kgGcCAwEAAaNT
+MFEwHQYDVR0OBBYEFNPm0Yesg8sMVK3OhDMCIGW5FumoMB8GA1UdIwQYMBaAFNPm
+0Yesg8sMVK3OhDMCIGW5FumoMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQEL
+BQADggEBAFXlSpQZKB3lrcE+hbBfP4ToamIMv6Jn41tlPm0saV1Wqr+p3pAidRlu
+tEZEVC20V5IUVT/k3BGZLtPmrQQHfHHVNeQNwfPQpa/DeVWdSoXSfPxFiv4nIDEy
+orYGse/TDWGloID5NkDIz/WKSBIF0lurKnScC0sBJxyyW3UvTf4Fkf+fIS3ovqFN
+2k0zs9DRPu1NReuRc+BxHqCNq1z+mjmK7Z2aSDl7rSox1tFUEl2QTNDrjFx7Mkxq
+ObWp7uHSdOQHauEwQ/PR28CTjHisMtYsoC4aLP0E6tiAv+/97VRj0UA/T1LcYcQ9
+dRnQwgp7I0aeV2cOlNfPPTqlWLNhhKo=
 -----END CERTIFICATE-----
diff --git a/tests/testdata/certs/pilot/ca-key.pem b/tests/testdata/certs/pilot/ca-key.pem
index 1f54c49558..ee9ca2124a 100644
--- a/tests/testdata/certs/pilot/ca-key.pem
+++ b/tests/testdata/certs/pilot/ca-key.pem
@@ -1,27 +1,27 @@
 -----BEGIN RSA PRIVATE KEY-----
-MIIEpQIBAAKCAQEA3vvGevQfvVK4EQYEjRqZOaW+PByCpkWYDDlzak4Kzd5SIlLP
-e1BhNF9XNpYE6vASaftGtAfwsnkRVCbs+uD5oJPuSh4FDIMb86qTaGlfUCsIuePA
-SKV1Q6UNKD4oE0mIiZEWExva+eUU9/OQ7lOxSTTU46dBNiblczOFDQK6pfey1wAA
-Ky4obGTcFfevnMeZwUficGluH2lvQNjZBn434HpDF7qjz2ptPyC1Zgs3VvfIaGfZ
-YZ4WRC1q53fpaPaNFLFajY8F7zFzyoMEUk63TMeD67Q+3u3eYKsEhnjkVYSpfToU
-66lJ7scXZ3tT/2kkZJkt72IAn/n1ZE6WeIns9QIDAQABAoIBAQCk3tJ1jNV+/Y99
-fs6UqksK9cK/3XC4khKHT19IBaY1a6GdIU+p5NfKiogIo0nFkHZ9CD7R1GEjBSNL
-7nAgW4IXOenhwWqIQtTDCKdvdyHzO657C6vbywL2Xi5pR/QHtsIOvUjuHhV92eT0
-JJJPtXALKdYS8VLlQEM9N6Wz5gWtuiLiSg5cP2FaRf/JukEFqomF8y3U8bFRWXMx
-ENzl01ueady2wl1Y8iyrsmj3R9BR4tcTJkwHf5pUU7VfGuYpXS/fPXcguez4CLhV
-WyLEn96MlWF1XktpTjTTDNgZILPMR3Tw7EjiHAelBPIFzgnuXyfW5vITQ/+eReGq
-i83ec0BhAoGBAP7lXPeVbRIXfbfNJ8dlA2GOvUGSSO7g1pG2YURs165DEga70mbR
-xM4DTX73PNu1pvNe6uwQ7Mw9gPvdMGKzRi3m16LyUE1i0MIrRJIvrCfpVEyxmieZ
-KkKfI4ch2JLdL0hPwGw9y6WCT0gpbSfUrZaVePfRi4HEUrZPGmUORltNAoGBAN/z
-Bs+O7LYbuQabFE7Qlfs9PGdVtKXWNr9fPbR8JhKsgOgo9mQUDgm1aeWpgyfyovvL
-IAYewQMrnhzwAApBggXBlrdAXwnwYcUGtwfLWxyHEu1KzuSzPGIYd/esoTjXNHRe
-b+xdAmiA1hXHC0tvXo455E1JbYRYfSAG4mYeB3RJAoGBAP64jtMMDFdMCmInLl1g
-YKSZVB+25BNztHkg3MrYSiDm/ZilEXmWnymQu1zqDXMycIg8jl8h+sGPpNoNDFFV
-/gfY7Bz2MbFj0mJ3gqrj4RJxZcDOqtJviX/SMFrWQ+tgBeQpbo28o0nekwP2AhMP
-wfj9nkloqhUf79gn7KSYQ07xAoGAYjxs5GdBlrCafTdRtTDyWX8HHB0QXR6WbWQN
-4ToESXS6Jxyw2D2j2sZlaYTuL3vT2rfDmGutJwLfHLp6igUqT/QXdNMb3XRArvi/
-GQAvO1hVCRojn1awSieJmG91WUWtkM5MMlaqU8UQJAVoAvMbezAR4YUwMTnPyVi7
-vye1BWECgYEAoQF8svL52X/pMIxBPUhAJUjYc/EUOwNtdwigE7dreF1q2n0RCduZ
-zKr/AQZy3BnqwSdSAHjOU4NWoRkznOCdREPbzmWUAqtumvQLoU35bQmaDIPjadvS
-UcIDgY6VjLaPMJ5Cp4P/lrbkfIo7BxP58uO++/Ota+IB4dTyodrcO48=
+MIIEogIBAAKCAQEAsPtybkCo2xdOIB6b9bHYfbp1it6Hk5WDkO1OQBs3uV+hP3y3
+0wU/lyCbMdwAp9pDRFJP7u2SnyPQgeC9m28lgcBgI2Opdui4qqAfaR08MCOvJa1Q
+iT3mZ316/3VbnQsoQRszPGYFNN2w2hnscO4JhVJ96eiDHNVSOLguF4ZxK+cNnlA2
++iiDv1KafubSqARZdmZ4PiltDmmFFMTYnPdXNOzOvkzOLub3M8S9HwwtByCBWdaY
+d0JrBU8ZmsFtZ8C0lcAU83A1P0j05cPZC5uYSO/giBOfG/Dt+j7XcJcoPanwVhin
+JlFoHBP/D0RzISrlMD4eyVabm7CqLoTOf2SAZwIDAQABAoIBABlWtshwO3y6tnXE
+zCF+J/fARWnveEu6AU8zmpt1CiJhPa3gRlVREAysH2whCnr9tdnHSyhzHSXps8WU
+jrD9Vcpg56LJdw7OPPo42KZtCBwAT+g+CyTLFvzEK2L9w3MVkNrZcRl5pc8u2k4v
+nfmBKx3KdLqTjooHRLFYNUkq93MIcW5TsfEgO5KEUu/ffFWpCYrz9WnVEpICdFcl
+maHUiyvnaCX7o7YSOj5rKWwogHbASQa6ZmHZYLQP6h9Fcs/cOpXwQWp20fKoaNQo
+a9KZeqCLTGy2vbr1qm0+1dhOw0Bi0/VakgXC0uXPRVvLeTka9PJydUhkERtRAzI/
+dC8x6EkCgYEA6AeDgdXE+sAQWBVw/griF7Nv5AxWEI+2BmuTHqj3t1CcdwXUnZVR
+t8W53Q9FerWoXhnm+5Qnx3YcuigAG57maRj6nb+pjw9rsESaiX0CfX9p/6VMQL4K
+48XNV7Xilc0TxmhbZ67AT2slGKuZjdhmBYKQufQoh6hQDAGLmRcKsJMCgYEAw0QZ
+qquSvl7H4YzxxfVppua+zTj85yWXyubNP2lk5oQtvvNFXoaYbmvnP1qP3gTyPKZ0
+j8/ERZxUm8RKJLQ5sVTqJvEytRkSrPkWwjOElNM85XEclvzsMgOEYGdyefZozzTa
+RGV0blEADn594KhWe9ZmfeIMV5NkJpXjO9QsGV0CgYAXc6r3FIQ0U/VS8dfZ1X/E
+pX5JrnWEdwIO5BCaqLlg7eTiotQ6mT0JNhLzJPgAzxuVZ0jRZMgggO+mYSShpg3j
+0YKDLzwUGZro1gZInzoBNT+hdzOd15eCLhvzIY8lVPUKuMGEWUVWwZ2eQLnkfFrT
+c77tDKCMFZcBn6ZcctfBwQKBgFfuKsC4UnR3qfrfxWAni1QlRHaV2w0b2Kb+GB8e
+DtnfW9pJIkvWwH25CYF1/qM0BwFY09SO8wuFwnQCiIDOx8wbOPYgV31FtSs/YAhD
+ushp91o6/WzgqFw9STdAXkSducTVQBTcuvmScDwWqgioxYkumyjEyloFfWwCwzkF
+6nXhAoGAbql6W/Oki3WFHlLVf5kU5TqpnLTxAtEttAgaLXLTpmrrOhJU870Qtjne
+ajWdo+dCay+DSE2n2h20oO5BPPTxm2zR++hjIJkONQiMWEw4w8mp+R17R7YXrsfO
+i3frmC56+pp28UPaEj3lJI4bjL5uYFEaBtJkd9unDVdJirBpVcc=
 -----END RSA PRIVATE KEY-----
diff --git a/tests/testdata/certs/pilot/cert-chain.pem b/tests/testdata/certs/pilot/cert-chain.pem
index 21b29b3716..f978e87903 100644
--- a/tests/testdata/certs/pilot/cert-chain.pem
+++ b/tests/testdata/certs/pilot/cert-chain.pem
@@ -1,22 +1,22 @@
 -----BEGIN CERTIFICATE-----
-MIIDtjCCAp6gAwIBAgIUaAVaH1ssljSp51Hm0bAEJUVf4dIwDQYJKoZIhvcNAQEL
-BQAwGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDAgFw0yMDAzMTMwMDUwNTBaGA8y
-MjkzMTIyNzAwNTA1MFowGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDCCASIwDQYJ
-KoZIhvcNAQEBBQADggEPADCCAQoCggEBAMUC4n6HeRwM7p3yxezoU/eu37hJ2zF2
-HazR2PP5Z4X+0M+deLeuyyoleqnQiuBq0taCD/4r6SCFltpVfrYtDgEcdkl86p+a
-YbTiFN+ICCNGSBNFVUzbdwndUIBXIl7N714BuToDDbbxUvDSytlPAlzCz3rrEBHr
-XPwi6BAiZ3taOWQ55hmMHmYuE7/8pJ9Q0xz+MCtPCn0WR3Vl1h44JJZhj6kMZPqn
-ZlJA9PgzPGHWEhoaBKxAhnDjd2ZKytbdAsaXLBzGzjHCFb4LGlnP/BXwf5CpMVfG
-v5q0UQ7kmYDX5QDRduxRMOvhK00TiWRH43b/Q5TN6p3IXFCS/QE/AvkCAwEAAaOB
+MIIDtjCCAp6gAwIBAgIUFad49XzcT8c0W+ltJRmkcmncz5UwDQYJKoZIhvcNAQEL
+BQAwGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDAgFw0yMjAzMDMyMjM2NTFaGA8y
+Mjk1MTIxNzIyMzY1MVowGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDCCASIwDQYJ
+KoZIhvcNAQEBBQADggEPADCCAQoCggEBAMQLuzWwb1Tg6u2Rl57RZQdIeHgwReea
+YDeyyURv1zJOzrZ6B6O1BHCRE0ekec46QBij2IGu6aTwtEKF99dYz2rBo1rzORo8
+b79x/aq4b5pu9Wn0LsYPvInGiKacDQD1Ngpe/XokM5obxKvgXN0vcevcYjHJoWBu
+LOmQ0wj/W+AhrV1A21q+OIi06lTBymbfqpcZH2GbwD9MvzX0xgdf6a09H8WgFcCS
+mCT9rHrXKgc+0aatxiNJrNzB/lAE5kXWzRiQ7a+7ImSVcq0kheaYXXGsO/+shTDM
+5l4+cGtvHz6t1RRe9lNzvYAWQsqNAHeG+ewNu9Z3AyQlLzpnKV22e48CAwEAAaOB
 9TCB8jAJBgNVHRMEAjAAMAsGA1UdDwQEAwIF4DAdBgNVHSUEFjAUBggrBgEFBQcD
 AgYIKwYBBQUHAwEwgbgGA1UdEQSBsDCBrYZFc3BpZmZlOi8vY2x1c3Rlci5sb2Nh
 bC9ucy9pc3Rpby1zeXN0ZW0vc2EvaXN0aW8tcGlsb3Qtc2VydmljZS1hY2NvdW50
 ghNpc3Rpb2QuaXN0aW8tc3lzdGVtghdpc3Rpb2QuaXN0aW8tc3lzdGVtLnN2Y4IY
 aXN0aW8tcGlsb3QuaXN0aW8tc3lzdGVtghxpc3Rpby1waWxvdC5pc3Rpby1zeXN0
-ZW0uc3ZjMA0GCSqGSIb3DQEBCwUAA4IBAQA3eFU1GVM/Dib4O04U4uEtwUQgqPzG
-qXTr653v/cz2669ibdBYeEioePiXlOvvm6kvNiLaSoEmb5t7bCgFQez4IDFZRRuk
-GraeBOnjzgJjbK6FQI0sNQjM1AjkC7sIMu1htF4UhLLgSecP+GSxI66tYtiMaNOz
-2Js8fvHbik35uIRWWXHGuIBnVLwmUZmHQ08bJyEy3qy0v3t6vU22GViXb+ZuQIYH
-c23eR/Q6m9feDi+Ela0rc/WS7EdJdZ7DJXvL/RGs5Dw9JOulIV1r9BLzGv4m/TzT
-5OQgUbe3yxdgm7/NtsA0heQkFRBjzV5x0kftFrQBxG/ZI17zvaDAEP+C
+ZW0uc3ZjMA0GCSqGSIb3DQEBCwUAA4IBAQAZitZ4wpn658Zqs4VlDVHmPTm+0Ui0
+0afk6aVJPlgA2r9F8riV1oyDHG5hs+tYXaVezU9zmQilMrzC5GdLYecn75FQ5Dd5
+Nc7GixcbgxBlVHWv5zP8gHd9giCFLWnrbx5VLCOZP8NPYoSXFn1htD1u7hd0ungc
+cT4tmycfH/0RbguAwSOL/ju7/REEwLwJAcidS2Q8JOTVPvJjxtEBDflDqnM2mIYO
+8lJOZcjZf4jJ8oNzMHv30ogkGTyr/m7hns1h3oUtjIarHgkG8ZlfOOBTExugfSQW
+E2AM1dV2WjiRZDxPW/Nc2F3oqgjE1hJtUmLQNedHqB3gzg0F+Jc18Bsc
 -----END CERTIFICATE-----
diff --git a/tests/testdata/certs/pilot/key.pem b/tests/testdata/certs/pilot/key.pem
index 41a677c8ab..4763e56719 100644
--- a/tests/testdata/certs/pilot/key.pem
+++ b/tests/testdata/certs/pilot/key.pem
@@ -1,27 +1,27 @@
 -----BEGIN RSA PRIVATE KEY-----
-MIIEpAIBAAKCAQEAxQLifod5HAzunfLF7OhT967fuEnbMXYdrNHY8/lnhf7Qz514
-t67LKiV6qdCK4GrS1oIP/ivpIIWW2lV+ti0OARx2SXzqn5phtOIU34gII0ZIE0VV
-TNt3Cd1QgFciXs3vXgG5OgMNtvFS8NLK2U8CXMLPeusQEetc/CLoECJne1o5ZDnm
-GYweZi4Tv/ykn1DTHP4wK08KfRZHdWXWHjgklmGPqQxk+qdmUkD0+DM8YdYSGhoE
-rECGcON3ZkrK1t0CxpcsHMbOMcIVvgsaWc/8FfB/kKkxV8a/mrRRDuSZgNflANF2
-7FEw6+ErTROJZEfjdv9DlM3qnchcUJL9AT8C+QIDAQABAoIBAHSfN4plEZk8Qoh3
-bRjHkuUcvMxv0/RckkZ6Vm5eJh0Y8WzlL8+0Bgol843QqFUehz558ZOpVLBuxBTs
-5TxcQV0BBziLXKwsDEiySF7mPML82K+nrE354GaTfxLqAauTRm4koQwpQJYaFtn+
-PhwTfDeDDqUZvwKMjBepZiwe4ZkPjqFQGUBNPMXmtL6CCIADC985md44DqHxPG6p
-vCZcNPXSCxLrnMBUXdNrA+UUpk6kfPyMANARPOvw1XNzC319gpTnM8b83HZou8cx
-Lk/NwGpn+mHku7IbCHL5dhgxeXgTTTxGmnKZ7J5agaBMNuyDMISGzfdlVgQhR5Wa
-RTOQVH0CgYEA6txxZ1DHCt0j2otehxOZpzXEWcf2OabpSscQAQUTk+RaxYminj2s
-ABdiQBcPZNVqI6E1awMQAyvTa78SKPzXVM2rscMtkda9hvgI9WD9bDz3d686dlMn
-p0RYau6PPQvXK6Ey3+bBh/IKCbaJVLpmPM0rYfTzJuC0wGSQeK9qLmcCgYEA1r5S
-7yHxUGiBNxROCf/2d8SIcQ06BSvvOEuM4XhmjQ1+B8TSC8MNWNX5u66iyXOa26OZ
-s50MIjvA3uRKLBYNJiCANBrJfw+XpLpmwLkoL7PAxVo6rOeAt5lbaQCaS1lvQiSO
-B18QjzFQx65L6xUSSK3Aduahk6gfnTj/tQNhp58CgYBl4DGg6z0rzTkcN5BNb4cb
-LQt7Fm4MHAbeVPLwPk97u3ZUcZuEoKPXdk08F8SprJlZ0zfkr3NmrMAM4Njzxplx
-Q7XSaiN+O3QC89yO8/950AhYdj4ZzYBJ4i+oUsr3ZAd1+Xqj/Oi+G5OXGwL/Kcvn
-W+gRmbtPhfFh8z1n3E9myQKBgQCyzgyuMtwEsM+Otih4qRlnBkz5nSLHdZqdjELE
-5oS1RBvBij+/LW0qoBhuDu5awRibc7CiXszgoFJrBTlBVicirQMy328RjkEdF1xB
-cqqkdY64hPXATI7f0SDM4ISY4uIB+AAO4TgfkteJPHbYSlZMD+jm2AiHswCH4BMC
-fMtOTwKBgQCO2szi3uYa7JzKQaJefrAVP6wZdtoPgRu6B/5vqeEazWv+eSiMavVs
-NSjnTgDRJTcwRL1fxlmNcpHJ9GIt6TDJ06yRMhXs0Xf8xl8lEO0IvQQwwV4ig738
-4UQI1B9uGKZr5ioU9i8c5UxarcRBUvN1RjuG/IIHa0J3HsRx9Ow2Xw==
+MIIEogIBAAKCAQEAxAu7NbBvVODq7ZGXntFlB0h4eDBF55pgN7LJRG/XMk7OtnoH
+o7UEcJETR6R5zjpAGKPYga7ppPC0QoX311jPasGjWvM5Gjxvv3H9qrhvmm71afQu
+xg+8icaIppwNAPU2Cl79eiQzmhvEq+Bc3S9x69xiMcmhYG4s6ZDTCP9b4CGtXUDb
+Wr44iLTqVMHKZt+qlxkfYZvAP0y/NfTGB1/prT0fxaAVwJKYJP2setcqBz7Rpq3G
+I0ms3MH+UATmRdbNGJDtr7siZJVyrSSF5phdcaw7/6yFMMzmXj5wa28fPq3VFF72
+U3O9gBZCyo0Ad4b57A271ncDJCUvOmcpXbZ7jwIDAQABAoIBAB3Ttxj1Dgs46SVZ
++FqGezt/cCzBE3GoUClJP0v1RCV4TgT/Tfg/SzyhJfDJemrA/A2DqCYa6ZtZr+NL
+JKkT7wLMEUF+LdrrBQl4m2KFCx3Q8e+74f3qWQIRPaszpNrXc/BiAWi650r0HJkV
+2qsgU9HRxvLSPYZBXP3wNdtxlpXj5Q8aPAfvfUy693K9laj7wOHtZ1ZGWQXMwC+K
+3cVqWl862cDnzhSdI+mZYvN3i7TjCyrmmS5BTaKGYeMTeHGkwfSfANwAJak1DaTT
+rpZXLJ83FbaxtlpqAGPFg1SXx/K8UjyeUd2HItCKAYBRAebYyog0AsZcVih42WVK
+Ely8XqkCgYEA/0n/FZM1HsS+qplNcpg3CEihJxc5LkO/hLuGvoEfTQYUVBeRdt0G
+FTZ5IzX6Yn9cRTR2JzU749qzJrg3CBDyQzjdIztR49uyefC8fRe/yANeSSTjucjp
+mPgq7TopXVTXBKCfw1SPiQVC/cWEUTG8uymctANNEFG2l5Kzd39Ugm0CgYEAxJd/
+nnEaGptqRGmafCXQGV3+jHDAMoVt5+1mD/A7vT6bdUqoLhCGXJs/bp9nSeUDOxXS
+Jz1F1dmEf2OzlvUCbxXthTXrWG5fSomw40wKccjkORsJXVYPi2H4hqfit5rOZMz6
+HBKBtYCUucDNyCen9cRnjqhWBKuwZkD9+t242GsCgYBbO5mSn0ojldJB64l3A5jf
+N/2Hl9ASjZmbGN1dpY2HHD4Vk1qj7Pcoi5AdGkRCARv7i3USK/8cPeM6Wikok83Q
+Yjj9R0zLro8I7xSda+hY1YNThoBvu7JnvYCxyNBRj2lMOQ7/Ym/tWtkHEhhoX+r/
+xC5BUtCunVUws/e5I+MSvQKBgEKpmLXSZmalUriWErOMYu9nJxzaUuIOysD9iEFZ
+VfiLFU4FndfY9KbMYgsR9IVjKeBMOvHhXh/w3XQRC+HhfQzMI7mrGf3nl1qALEI3
+E8eXtoQCvf8vYqAB/bMzlLZrwRb/RsRkJiW0zznZvi3j/JljGzaNsHVu/qb93OHH
+c5m3AoGAP94yjweizoPEcYJuf/J0KxDfxRepS4Mc/TG+vx5UcDwOS95Yf3AAWGdP
+EsNKe1nLQBo8Zxv3VymTEEdZ4TduzSvwbkP3UUMwU2/O4qnB35yJsXonc6BrteUK
+3Bq1QdI3I0KomJy/FEhqeKtKeeOOJIpLZdAQp+KuXYoneHh96Kc=
 -----END RSA PRIVATE KEY-----
diff --git a/tests/testdata/certs/pilot/root-cert.pem b/tests/testdata/certs/pilot/root-cert.pem
index 71062b7f8c..6f9c872974 100644
--- a/tests/testdata/certs/pilot/root-cert.pem
+++ b/tests/testdata/certs/pilot/root-cert.pem
@@ -1,19 +1,19 @@
 -----BEGIN CERTIFICATE-----
-MIIDEzCCAfugAwIBAgIUHu+RNpObSStNAQtgGqFPV70CNVEwDQYJKoZIhvcNAQEL
-BQAwGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDAgFw0yMDAzMTMwMDUwNTBaGA8y
-MjkzMTIyNzAwNTA1MFowGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDCCASIwDQYJ
-KoZIhvcNAQEBBQADggEPADCCAQoCggEBAN77xnr0H71SuBEGBI0amTmlvjwcgqZF
-mAw5c2pOCs3eUiJSz3tQYTRfVzaWBOrwEmn7RrQH8LJ5EVQm7Prg+aCT7koeBQyD
-G/Oqk2hpX1ArCLnjwEildUOlDSg+KBNJiImRFhMb2vnlFPfzkO5TsUk01OOnQTYm
-5XMzhQ0CuqX3stcAACsuKGxk3BX3r5zHmcFH4nBpbh9pb0DY2QZ+N+B6Qxe6o89q
-bT8gtWYLN1b3yGhn2WGeFkQtaud36Wj2jRSxWo2PBe8xc8qDBFJOt0zHg+u0Pt7t
-3mCrBIZ45FWEqX06FOupSe7HF2d7U/9pJGSZLe9iAJ/59WROlniJ7PUCAwEAAaNT
-MFEwHQYDVR0OBBYEFC1YD3F9RMLaW1xMuuQN27SmXzsMMB8GA1UdIwQYMBaAFC1Y
-D3F9RMLaW1xMuuQN27SmXzsMMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQEL
-BQADggEBAGqzLzTnusoOPF+tQ4jbhcXDCMvpDCCiAQVkkaorNGMWg2ny39vm3ymc
-HBhqWNNbeyZTj8hl7ffCoeUChZnIFkLO1ev3DVbLz179IP6dhqfDkuLhc/0g5Lfs
-FcNedAJh5TConI+x7dH1W2opPX4hXiiJBzcpPmfttD1EYES2VH85cPp95goUJwRp
-kMPAS5WiBLPv0c+0Msppe4iqxwuZZtJKRoHRDbvrHonMT1FmdOoEJ8xjeTxB8pjc
-tp5dA30dGNAxB7tOrzqmIidjXjYaxZGXKoC6BViw9ISfX9WNqVdvZVMJ7XwL6gu5
-Dk0yYXcMuGwMkMz+vJpypIBZAaCFES8=
+MIIDEzCCAfugAwIBAgIUf7oS8NPqTONgWOz1s8w4I6QWrv8wDQYJKoZIhvcNAQEL
+BQAwGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDAgFw0yMjAzMDMyMjM2NTFaGA8y
+Mjk1MTIxNzIyMzY1MVowGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDCCASIwDQYJ
+KoZIhvcNAQEBBQADggEPADCCAQoCggEBALD7cm5AqNsXTiAem/Wx2H26dYreh5OV
+g5DtTkAbN7lfoT98t9MFP5cgmzHcAKfaQ0RST+7tkp8j0IHgvZtvJYHAYCNjqXbo
+uKqgH2kdPDAjryWtUIk95md9ev91W50LKEEbMzxmBTTdsNoZ7HDuCYVSfenogxzV
+Uji4LheGcSvnDZ5QNvoog79Smn7m0qgEWXZmeD4pbQ5phRTE2Jz3VzTszr5Mzi7m
+9zPEvR8MLQcggVnWmHdCawVPGZrBbWfAtJXAFPNwNT9I9OXD2QubmEjv4IgTnxvw
+7fo+13CXKD2p8FYYpyZRaBwT/w9EcyEq5TA+HslWm5uwqi6Ezn9kgGcCAwEAAaNT
+MFEwHQYDVR0OBBYEFNPm0Yesg8sMVK3OhDMCIGW5FumoMB8GA1UdIwQYMBaAFNPm
+0Yesg8sMVK3OhDMCIGW5FumoMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQEL
+BQADggEBAFXlSpQZKB3lrcE+hbBfP4ToamIMv6Jn41tlPm0saV1Wqr+p3pAidRlu
+tEZEVC20V5IUVT/k3BGZLtPmrQQHfHHVNeQNwfPQpa/DeVWdSoXSfPxFiv4nIDEy
+orYGse/TDWGloID5NkDIz/WKSBIF0lurKnScC0sBJxyyW3UvTf4Fkf+fIS3ovqFN
+2k0zs9DRPu1NReuRc+BxHqCNq1z+mjmK7Z2aSDl7rSox1tFUEl2QTNDrjFx7Mkxq
+ObWp7uHSdOQHauEwQ/PR28CTjHisMtYsoC4aLP0E6tiAv+/97VRj0UA/T1LcYcQ9
+dRnQwgp7I0aeV2cOlNfPPTqlWLNhhKo=
 -----END CERTIFICATE-----
-- 
2.35.3

